<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>pathogen_genomics_week1</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="assets/styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">COG-Train_Resources</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-file-lines"></span>
     
    Course Materials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">Virtual Courses</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Viral Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="vbioinfasia.html">VBioinf2022 Asia</a>
        </li>
        <li>
          <a href="vbioinflac.html">VBioinf2022 LAC</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">MOOCs</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Power of Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="power_of_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="power_of_genomics_set1.html">Week 1</a>
        </li>
        <li>
          <a href="power_of_genomics_set2.html">Week 2</a>
        </li>
        <li>
          <a href="power_of_genomics_set3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Swab to Server</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="swab_to_server_home.html">Home</a>
        </li>
        <li>
          <a href="swab_to_server_week1.html">Week 1</a>
        </li>
        <li>
          <a href="swab_to_server_week2.html">Week 2</a>
        </li>
        <li>
          <a href="swab_to_server_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Making Sense</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="making_sense_home.html">Home</a>
        </li>
        <li>
          <a href="making_sense_week1.html">Week 1</a>
        </li>
        <li>
          <a href="making_sense_week2.html">Week 2</a>
        </li>
        <li>
          <a href="making_sense_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Practical Guide</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="practical_guide_home.html">Home</a>
        </li>
        <li>
          <a href="practical_guide_week1.html">Week 1</a>
        </li>
        <li>
          <a href="practical_guide_week2.html">Week 2</a>
        </li>
        <li>
          <a href="practical_guide_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Pathogen Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="pathogen_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="pathogen_genomics_week1.html">Week 1</a>
        </li>
        <li>
          <a href="pathogen_genomics_week2.html">Week 2</a>
        </li>
        <li>
          <a href="pathogen_genomics_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Hybrid Course</li>
    <li>
      <a href="scvo2b4b.html">SARS-CoV-2 B4B</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">In Person Courses</li>
    <li>
      <a href="ECPG.html">Pathogen Genomics Capacity</a>
    </li>
    <li>
      <a href="IGM_B4B.html">Bioinformática Decodificada</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Webinars</li>
    <li>
      <a href="Community.html">COG-Train Community</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-book"></span>
     
    Team
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cogteam.html">COG-Train Team</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/WCSCourses/COG-Train_Resources">
    <span class="fa fa-brands fa-github"></span>
     
    Github
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="introduction-to-viral-surveillance" class="section level1">
<h1>INTRODUCTION TO VIRAL SURVEILLANCE</h1>
<p><img src="images/OC5_3-13_illustrative.png" /><br />
<sub><span class="citation">@Canva</span></sub></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="glossary" class="section level1">
<h1>Glossary</h1>
<!-- 1-4 -->
<ul>
<li><p><strong>AMR:</strong> Antimicrobial Resistance, which occurs when
a microbe or pathogen genome evolves or acquires genes or plasmids that
confer resistance to an antimicrobial.</p></li>
<li><p><strong>Antimicrobials:</strong> agents (e.g drugs) that can
interrupt the life cycle or replication of a microbe or
pathogen.</p></li>
<li><p><strong>Endemic:</strong> refers to a disease that is prevalent
in or restricted to a particular location, region, or
population.</p></li>
<li><p><strong>Epidemic:</strong> is an increase in the expected number
of cases of a disease in a particular population and area.</p></li>
<li><p><strong>HIC:</strong> High-Income Countries, which is defined as
a group of countries with highly developed economies by the World Bank
classification.</p></li>
<li><p><strong>Limnology:</strong> the study of aquatic ecosystems in
inland waters such as lakes, reservoirs, rivers, streams, wetlands and
groundwater.</p></li>
<li><p><strong>LMICs:</strong> Low-and-Middle-Income Countries, a group
of countries with less developed economies by the World Bank
classification.</p></li>
<li><p><strong>MDR:</strong> Multidrug-Resistant, which refers to genes
or pathogen strains that confer resistance to most or all available
antimicrobials.</p></li>
<li><p><strong>NCD:</strong> Non-Communicable Disease, a disease that is
not transmissible from one person to another. Instead, they are the
result of a combination of genetic, physiological, environmental and
behavioural factors.</p></li>
<li><p><strong>NGS:</strong> Next-Generation Sequencing, a
high-throughput sequencing methodology.</p></li>
<li><p><strong>ONT:</strong> Oxford Nanopore Technologies, a sequencing
technology company. Their sequencing platform is also referred to as
ONT.</p></li>
<li><p><strong>Outbreak:</strong> is a sudden increase in the expected
number of cases of a disease in a limited area.</p></li>
<li><p><strong>Pandemic:</strong> refers to an epidemic that has spread
over several countries or continents, usually affecting a large number
of people.</p></li>
<li><p><strong>Plasmid:</strong> in nature, this is a small circular
double-strand DNA molecule that can benefit survival or selective
advantage to bacteria.</p></li>
<li><p><strong>RT-qPCR:</strong> Reverse Transcriptase Quantitative PCR,
also known as reverse transcriptase real-time PCR. A PCR method to
quantify RNA copies present in a sample; includes a step to synthesise
cDNA from the template RNA before the PCR amplification.</p></li>
<li><p><strong>SEDRIC:</strong> Surveillance and Epidemiology of
Drug-resistance Infections Consortium, a genomics working group to
monitor drug-resistance.</p></li>
<li><p><strong>TB:</strong> is the acronym for tuberculosis, a lung
infectious disease caused by the pathogenic bacteria <em>Mycobacterium
tuberculosis</em>.</p></li>
<li><p><strong>Transposon:</strong> is a DNA fragment that can change
its position in a genome through cellular mechanisms.</p></li>
<li><p><strong>VOC:</strong> SARS-CoV-2 Variant of Concern, a viral
variant with mutations that confer a beneficial feature to the virus,
e.g. increased transmissibility or virulence.</p></li>
<li><p><strong>WASH:</strong> Water, Sanitation and Hygiene, a commonly
used term among stakeholders advocating for universal access to WASH,
especially in resource-limited settings.</p></li>
<li><p><strong>WGS:</strong> Whole-Genome Sequencing, a method of
determining the entirety, or almost the entirety, of an organism’s
genomic DNA sequence all at once. This entails sequencing all of an
organism’s chromosomal DNA, as well as, DNA found in mitochondria and,
in plants, chloroplasts.</p></li>
</ul>
<p><a href="assets/OC5_1-4_Glossary.pdf">Donwload the PDF here</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="pathogen-monitoring" class="section level1">
<h1>Pathogen monitoring</h1>
<!-- 1-7 -->
<!-- YT https://youtu.be/XYjayaMStRY -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/XYjayaMStRY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-7_transcript.pdf">Download the transcript
here</a></p>
In this video, Dr Josefina Campos shares her experiences and explains
the importance of genomics surveillance to monitor and control pathogen
outbreaks.
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="working-together-to-build-capacity" class="section level1">
<h1>Working together to build capacity</h1>
<!-- 1-8 -->
<!-- YT  https://youtu.be/oYTyTbed7O8 -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/oYTyTbed7O8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-8_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Tomas Poklepovich Caride from the National Center
of Genomics and Bioinformatics, Argentina, tells us what the
requirements for efficient pathogen monitoring in a laboratory facility
are, including storage space and bioinformatics.</p>
<p>A robust healthcare system and infrastructure are vital for
prevention, preparedness, early detection and response to pathogens.
Thus, it is necessary to build a robust, comprehensive, and integrated
surveillance programme.</p>
<p>Although the <a
href="https://doi.org/10.1371/journal.pone.0245457">efficient
implementation</a> of surveillance programmes needs to consider social,
economic, environmental, and technical conditions in the health
landscape, not all surveillance systems are suitable for all countries
or regions. Robust surveillance programmes depend on national healthcare
infrastructure at the primary and reference level, as well as logistics,
political and geographical realities.</p>
<p>The emergence of new diseases, conditions and events drive the review
of public health priorities and strategies. After the COVID-19 pandemic,
the incorporation of new technologies, including genomics surveillance
and <a
href="https://www.paho.org/en/topics/influenza-and-other-respiratory-viruses/covid-19-genomic-surveillance-regional-network">new
networks</a> added another <a
href="https://doi.org/10.1186/s40249-020-00757-5">level of
complexity</a> to pathogen monitoring.</p>
<p>The <a href="https://doi.org/10.20506/rst.38.1.2949">One Health
concept</a> has been increasingly discussed in the past two decades. It
leverages a global health security approach by improving coordination,
collaboration, and communication at the human-animal-environment
interface. One Health addresses global health threats such as zoonotic
diseases, antimicrobial resistance, and food safety among others.</p>
<p>To extend surveillance approaches, it is essential to engage and
apply multi and transdisciplinary expertise. For example, a limnologist
represents a valuable asset to better understand the ecology of vector
or waterborne diseases. However, anthropologists could be advantageous
in a field investigation, as the relationship between the community and
the water is completely different according to history and cultural
backgrounds.</p>
<p>One of the major lessons learnt from the COVID-19 pandemic is that
multidisciplinary teams are key to responding to a global threat. The
scientific community working together with epidemiologists, public
health laboratories, stakeholders and decision-makers was essential for
a strong pandemic response.</p>
<p>In the context of genomic surveillance, bioinformatics and IT team
members must be included in the decision-making process, so that they
can rethink the informatics architecture and ensure that translate
genomics results to feed the surveillance systems into measurable
actions. We also need to consider how to develop a surveillance system
that engages with <a
href="https://doi.org/10.1038/s41467-022-28529-9">national reference
laboratories</a>, provincial or regional laboratories, academics and
universities, for all, will input <a
href="https://doi.org/10.1089/fpd.2021.0110">genomics information</a>
into the system.</p>
<p>Although access to core genomic facilities and bioinformatics
expertise is still limited in public healthcare facilities in
low-and-middle-income countries and many in high-income countries, the
need to respond to COVID-19 enabled the improvement of genomic
structures across the globe. These resources can be repurposed to
provide <a href="https://doi.org/10.1016/S2666-5247(22)00058-1">genomic
surveillance response</a> of all pathogens of public health importance
and also some non-transmissible diseases diagnosis.</p>
<p>Effective disease surveillance requires the collaboration of multiple
teams, including genomics specialists, disease reference laboratories,
bioinformatics, epidemiologists, and physicians; they also need to be
aligned with national plans, which can add another layer of complexity.
The challenge encountered by multidisciplinary teams includes not only
disease detection, long-term surveillance, data reporting, management
and analytics. They also must liaise with other stakeholders, including
industry, and need to be prepared for information dissemination,
logistics and continuous capacity building and training.</p>
<p>The pharmaceutical industry also played a major role in the <a
href="https://doi.org/10.1016/S2214-109X(22)00341-2">COVID-19 response
team</a> in vaccine development and scale-up production. Industry
capacity needs to be considered in the surveillance scheme of each
disease’s dynamics. For example, if we are implementing a vector control
strategy, it’s important to consider:</p>
<ul>
<li><p>Is the industry able to provide the insecticide chosen?</p></li>
<li><p>Is it available for local production? If not, can it be promoted
by the programme?</p></li>
</ul>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="what-genomic-data-could-tell-us" class="section level1">
<h1>What genomic data could tell us</h1>
<!-- 1-9 -->
<!-- YT  https://youtu.be/Yf0So-wfsaI -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/Yf0So-wfsaI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-9_transcript.pdf">Download the transcript
here</a></p>
<p><a href="assets/OC5_1-9_slides.pdf">Download the PDF slides
here</a></p>
<p>In this video lecture, Prof Philippe Lemey uses phylogenetic data to
show how genomic data can be presented and how to interpret it.</p>
<p>Note: HBV - Hepatitis B Virus</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="transferring-knowledge" class="section level1">
<h1>Transferring knowledge</h1>
<!-- 1-10  -->
<!-- YT https://youtu.be/A7VvZ5pe_Uk -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/A7VvZ5pe_Uk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-10_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Prof Christine Carrington from the University of the
West Indies, Trinidad and Tobago, Dr Emma Hodcroft from the Universities
of Bern and Geneva, Switzerland and Dr Richard Orton from MRC-University
of Glasgow Centre for Virus Research, United Kingdom discuss with Dr
Thanat Chookajorn relevant questions on how we can use genomics to
improve the surveillance of pathogens of public health relevance. They
discuss the importance of training, education and the transference of
knowledge to laboratory teams and academic researchers.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="laboratory-quality-management" class="section level1">
<h1>Laboratory quality management</h1>
<!-- 1-11 -->
<!-- YT https://youtu.be/DssHD73aKKQ  -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/DssHD73aKKQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-11_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Ana Filipe from MRC-University of Glasgow Centre
for Virus Research, United Kingdom and Dr Harper VanSteenhouse from
BioClavis Research, United States of America talk about quality control
and standards that can be used across a national laboratory network.</p>
<p>To ensure that laboratories provide results that reduce the risk of
errors and are both accurate and reliable, it is important to implement
a <strong>Quality management system (QMS)</strong>. However, in
resource-constrained settings, most laboratories are not accredited and
may only be partially implementing elements of a QMS. Introducing a new
test, particularly under outbreak conditions, may therefore come with a
high risk of errors. This step describes the key critical elements that
laboratories should put in place to ensure quality results in less
optimal conditions, such as during outbreaks of COVID-19.</p>
<p><strong>Quality Control (QC)</strong> is a mechanism that monitors
the analytical performance of the test when used with or as part of a
test system. It may monitor the entire test system or only one aspect of
the test. QC validates the competency of testing laboratories by
assessing sample quality and monitoring test procedures, test kits, and
instruments against established criteria. It also includes the review of
PCR results and documentation of the validity of testing methods. QC is
therefore a multi-step process with certain checkpoints throughout the
testing process: pre-analytical, analytical, and post-analytical stages.
In general, QC should be performed regularly to detect, evaluate, and
correct errors due to test system failure, environmental conditions, or
operator performance before reporting test results.</p>
<p>QCs that are commonly employed for PCR testing include
<strong>extraction negative control</strong> (checks contamination at
extraction phase), <strong>extraction positive control</strong> (checks
extraction process-reagents and equipment functionality),
<strong>non-template control</strong> (checks contamination at PCR
phase), <strong>positive template control(s)</strong> (check(s) limit of
detection and robustness of the assay). Commercial QCs for positive
controls are preferred, however, laboratories can use patient samples
with known viral RNA concentration, preferably samples with low cycling
threshold (Ct) values (25–30) for the target sequences of SARS-CoV-2.
Nuclease-free water or viral transport medium can be used as the
negative control. QC failures, for example, when a positive control
turns out negative or a negative control turns out positive, invalidate
the test results. Common causes of failure include contamination,
degradation of samples, and expired reagents. After investigating and
fixing the cause of the QC failure, the test must be repeated using
either stored or newly collected samples.</p>
<p><strong>External Quality Assessment (EQA)</strong> is a process that
allows laboratories to assess their performance by comparing their
results with those from other laboratories within the network (testing
and reference laboratories) via panel testing and retesting. EQA also
includes the onsite evaluation to review the quality of the laboratory’s
performance. It usually evaluates testing competency, the performance of
the laboratories, the reliability of the testing methods, and the
accuracy of the results reports, including following up on any
unacceptable EQA results with corrective action. One or more of the
following three EQA methods can be applied to COVID-19 molecular testing
laboratories:</p>
<p><strong>Proficiency testing (PT)</strong> is when an external
provider sends a set of SARS-CoV-2 positive and negative simulated
clinical samples for testing in different laboratories and the results
of all laboratories are analysed, compared, and reported back to the
participating laboratories. Laboratories should select PT providers with
a track record in delivering PT panels within their region.</p>
<p><strong>Retesting</strong> refers to samples that have been tested at
one laboratory and are then retested at another laboratory, allowing for
inter-laboratory comparison. A laboratory’s first positive COVID-19
sample should be sent to another testing laboratory, preferably a
national or a WHO reference laboratory. In the absence of PT, national
COVID-19 laboratories should send five positive and ten negative
samples, systematically selected, to WHO reference laboratories for
retesting. Similarly, sub-national COVID-19 testing laboratories should
send retesting samples to their national COVID-19 reference
laboratory.</p>
<p><strong>Onsite evaluation</strong> should be performed by experienced
subject matter experts, who observe and assess the quality management
systems of the COVID-19 testing laboratories across the three testing
phases. Onsite evaluation includes Patient Management, Biosafety
adherence, Quality control procedures, Staff competency, Sample
collection procedures, Standardised testing policies, Documentation and
maintenance of records. Onsite evaluation should be conducted at least
once every twelve months, but preferably every three to six months.</p>
<p><a href="assets/OC5_1-11_sup_mat1.pdf">Download an article about
Quality Management from the Africa Society for Laboratory Medicine
(ASLM) here</a></p>
<p><a href="assets/OC5_1-11_sup_mat2.pdf">Download Guidance for
SARS-CoV-2 surveillance in Africa from the African Union here</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="why-biobanks-are-important" class="section level1">
<h1>Why biobanks are important</h1>
<!-- 1-12 -->
<!-- YT https://youtu.be/3k11ydVZonU   -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/3k11ydVZonU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-12_transcript.pdf">Download the transcript
here</a></p>
<p>A biobanking system that collects, stores and archives specimens
(e.g. viruses or other pathogens) to facilitate the development of
diagnostic tests and evaluate diseases with pandemic potential can be
crucial to public health responses. But what is a biobank? In this
video, Gideon Nsubuga walks us through the Integrated Biorepository
facility of H3Africa at Makerere University, Uganda. You will find out
that biobanking is a lot more than a freezer farm.</p>
<p>Just like how a bank accepts (receives) and safeguards (stores) money
owned by other individuals and entities, and then lends (gives
out/shares) this money in order to conduct economic activities or simply
to cover operating expenses (value addition), biobanks receive and store
biological samples and health information. This is done in an ethically
and legally regulated manner, and biobanks share these samples with
researchers who can conduct research and investigate diseases using
them.</p>
<p><strong>What is a biobank?</strong></p>
<p>A biobank/biorepository/bioresource is a collection of biological
samples and health information. The purpose of a biobank is to process,
organise, and maintain various types of biospecimens that are shared for
utilisation in both clinical and research-based services (Figure 1).</p>
<div class="float">
<img src="images/OC5_1-12.png" alt="Biobanking process" />
<div class="figcaption">Biobanking process</div>
</div>
<p><a href="images/OC5_1-12.png">Click here to enlarge the image</a></p>
<p><a href="images/OC5_1-12_Alt-text.pdf">Download Figure 1 alt-text
here</a></p>
<p>Different biobanks collect different types of samples and
information. The types of information and samples collected depend on
the specific purpose of the biobank. For example, some biobanks are
specific to a particular disease, such as cancer. Other biobanks are
population-based and contain samples and information from people in a
specific population or region. The biobanks have become a crucial
resource for medical research since the late 1990s as they support
various sectors of research, such as the field of personalised medicine
and genomics. In fact, TIME magazine listed biobanks as being among the
<a
href="https://content.time.com/time/specials/packages/article/0,28804,1884779_1884782_1884766,00.html">ten
big ideas changing the world</a>. Biobanks allow researchers to access
biospecimens and data that represents a large number of people, and this
can subsequently be used by multiple researchers for cross-purpose
research studies.</p>
<p><strong>Biospecimens</strong></p>
<p>Biospecimen types that are available include organ tissue, blood,
saliva, urine, skin cells, and other tissues or fluids taken from the
body. The samples are maintained in appropriate condition to prevent
deterioration and are protected from both accidental and intentional
damage. The sample is registered in a computer-based system. The
physical location of the specific sample is also noted to enable the
specimen to be easily located when required. Samples are de-identified
to ensure donor privacy and allow researchers to analyse without bias.
Room temperature storage may be used in some cases due to cost
efficiency and so as to avoid issues such as freezer failure.</p>
<p><strong>Ethics</strong></p>
<p>Biobanks can only store and share samples if they follow the required
ethical guidelines. Some ethical issues around biobanking are the
ownership of samples, ownership of derived data, the right to privacy
for donors, the extent of donor consent, and the extent to which
information the donor can share upon the return of research results.</p>
<p><strong>How does a biobank make performing research
easier?</strong></p>
<p>The biobank serves as a library for researchers. Therefore, the time
and resources needed to recruit new participants for each research study
are greatly reduced because samples and the corresponding medical
information are already available in one place. By making sample
collection and patient recruitment more efficient, studies can be
performed more quickly and with greater quality control.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="metadata" class="section level1">
<h1>Metadata</h1>
<!-- 1-13 -->
<div class="float">
<img src="images/OC5_1-13_fig1.png"
alt="An interesting-looking graph with incomplete labelling" />
<div class="figcaption">An interesting-looking graph with incomplete
labelling</div>
</div>
<p><a href="images/OC5_1-13_fig1.png">Click here to enlarge the
image</a></p>
<p><em>Source: <a
href="https://foxhugh.files.wordpress.com/2012/02/graph_20060627_lg.jpg">Hugh
Fox III</a></em></p>
<p><a href="images/OC5_1-13_Alt-text.pdf">Donwload Figure 2 alt-text
here</a></p>
<p>Imagine coming across this interesting-looking graph in Figure 2
above, but not knowing exactly what it’s trying to show you. The heading
which read: “Competitive Eating Records” has been cropped off, and there
are missing y-axis labels which should have shown that the yellow bars
were the total weight of food ingested (kg) and the blue bars were the
rate of ingestion (kg/min).</p>
<p>Most of you will be familiar with the concept of “garbage in; garbage
out” (GIGO) in the context of ensuring good quality data. However, even
if you have the best quality data, if your data has no context, you have
no story.</p>
<p>So what exactly is metadata, and why is it important?</p>
<p>It is now easier than ever to access millions of sequenced datasets
deposited in public databases. This <a
href="https://doi.org/10.1007/s12551-018-0490-8">mining of data</a>
enables us to do extensive meta-analyses to gain new and unexpected
insights into the underlying biological mechanisms across viruses and
the tree of life.</p>
<p>However, this process is often hindered by a lack of accompanying
metadata. <a
href="https://doi.org/10.1186/s13059-021-02332-z">Metadata</a> comprises
any data that describes the sample type, collection procedure,
extraction, assay methods used, analysis with the chosen parameters,
filtering steps, quality control, reporting of software versions, as
well as any other phenotypic descriptions.</p>
<p>Most of the time, some metadata is provided during data submissions,
but this metadata is often unstandardised. A <a
href="https://doi.org/10.1007/s12551-018-0490-8">standardised format</a>
implies that there is a checklist which necessitates that a minimum
amount of metadata must accompany the data and that free text is limited
in favour of a controlled vocabulary.</p>
<p>The COVID-19 pandemic has especially highlighted <a
href="https://doi.org/10.1038/s41597-020-0524-5">how necessary it is to
have informed data</a>. Metadata such as the time and the place where
the samples were collected, which may have not been as important to
genomic datasets before, suddenly became crucial. Many repositories such
as <a href="https://www.ncbi.nlm.nih.gov/geo/">GEO</a> and <a
href="https://www.ebi.ac.uk/ena/browser/home">ENA</a> now require
research teams to use a metadata sheet with compulsory fields that must
be populated when submitting data. Extensive efforts have also been made
towards both manual curation and the automated assignment of metadata,
through natural language processing (NLP) and the <a
href="https://doi.org/10.1186/s13059-021-02332-z">use of machine
learning models</a>.</p>
<p>Describing the WHO, WHAT, HOW, WHERE, and WHEN of genomic data also
contributes to the findable, accessible, interoperable and reproducible
<a href="https://doi.org/10.1038/sdata.2016.18">(FAIR)</a> guiding
principles, to support the reuse of scientific data, and very
importantly, it helps to inform public health responses.</p>
<p>Although we acknowledge the importance of accompanying genomic data
with the appropriate metadata, we also have to be cognizant of the fact
that the collection, anonymisation, storage and access to this data have
to be carefully managed in a way that protects the donors. Submitting a
data management plan as part of the research plan has therefore become
an increasingly <a
href="https://doi.org/10.1016/j.jbi.2021.103900">standard
procedure</a>.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="the-ethics-of-data-sharing" class="section level1">
<h1>The ethics of data sharing</h1>
<!-- 1-14 -->
<p>Coordinated data sharing maximises the utility of data, which is
critical for the control of infectious diseases. Data sharing is
imperative during pandemics and public health emergencies. For example,
many of the major achievements towards the containment of COVID-19 are
attributed to the rapid sharing of sequence data across the globe. This
step highlights some ethical considerations regarding genomic data
sharing with examples from HIV phylogenetics and COVID-19 and presents
some mitigation strategies.</p>
<p><strong>Risk to privacy</strong></p>
<p>Data sharing is widely used in HIV phylogenetic analysis. Balancing
the benefits of understanding HIV transmission dynamics with the
potential risks of harm arising from an <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6450547/">unintentional
breach of personal privacy</a> remains a major ethical consideration in
HIV phylogenetics. Such concern is justified considering that each HIV
sequence is associated with a human being. While sharing HIV genetic
sequences facilitates HIV-related research, it may worsen the risk of
inferential privacy loss. These concerns should be viewed within the
context of HIV social stigma and criminalisation of HIV transmission,
which pose severe risks of social harm to research participants in the
event of <a href="https://doi.org/10.1177/1556264618809608">inadvertent
disclosure of personally traceable results</a>.</p>
<p><strong>Equitable data use and benefit sharing</strong></p>
<p>Apart from potential risks of harm to individuals, there are ethical
issues regarding the fair use of valuable data generated and shared by
researchers from low to middle-income countries (LMICs). While their
contributions might be acknowledged, the scientific benefits that accrue
from the analysis of the sequence data might be inaccessible to the
communities that contributed data towards that benefit (e.g. vaccines
not being available to those communities). This results in mistrust and
uneven vaccine access, as witnessed during the <a
href="https://pubmed.ncbi.nlm.nih.gov/18618060/">H5N1 outbreak</a> and
recently the <a
href="https://doi.org/10.1136/medethics-2020-106858">COVID-19
pandemic</a>. Furthermore, researchers based in LMICs should also
receive appropriate credit for their contribution to eventual scientific
publications.</p>
<p><strong>Mitigation strategies</strong></p>
<p>Several strategies can mitigate the risks posed by sharing pathogen
genomic data. Before sharing, protocols for the deidentification or
anonymisation of data are required. Only limited information should be
published with each sequence, and additional information should be
released using a strictly controlled access protocol. Controlled digital
systems that facilitate collaboration instead of data sharing could be
prioritised.</p>
<p>If data are to be released for research purposes, the study should
meet the ethical principles outlined in the <a
href="https://bioethicsarchive.georgetown.edu/pcsbi/sites/default/files/Emanuel%2C%20E.J.%2C%20Wendler%2C%20D.%2C%20Grady%2C%20C.%20%E2%80%9CAn%20Ethical%20Framework%20for%20Biomedical%20Research.%E2%80%9D%20The%20Oxford%20Textbook%20of%20Clinical%20Research%20Ethics.%20Eds.%20E.J.%20Emanuel%20et%20a">Emanuel
Framework</a>. Furthermore, researchers should comply fully with the
information provided in the consent documents, which specify what and
how data will be shared and reused unless otherwise authorised by local
or international ethics guidelines. Material Transfer Agreements between
institutions should also provide clear statements on benefit-sharing,
especially how products from the research (e.g., therapeutics or
vaccines) will be shared.</p>
<p>In conclusion, further ethical-legal scholarship (including empirical
work) is warranted to determine an optimal ethical framework and a best
practice model to guide HIV genetic data sharing and use, including for
molecular HIV surveillance. Lessons learned (e.g., risk/benefit
determinations) from such sharing during the COVID-19 pandemic should
also be reviewed in this process.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-share-pathogen-genomic-data" class="section level1">
<h1>How to share pathogen genomic data</h1>
<!-- 1-15 -->
<!-- YT https://youtu.be/VxqH686g4eI -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/VxqH686g4eI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-15_transcript.pdf">Download the transcript
here</a></p>
<p><a href="assets/OC5_1-15_slides.pdf">Download the PDF slides
here</a></p>
<p>Dr Euzebiusz Jamrozik from Ethox Centre, United Kingdom shares his
experience in ethically using data to improve HIV phylogenetics
analysis.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="data-sharing-in-a-public-health-emergency"
class="section level1">
<h1>Data sharing in a public health emergency</h1>
<!-- 1-16 -->
<!-- YT   https://youtu.be/tbjo1cLNgQI  -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/tbjo1cLNgQI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC5_1-16_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Ewan Harrinson from Wellcome Sanger Institute and
the University of Cambridge guides us through the differences between
data sharing routinely and during a public health emergency.</p>
<p>It is widely acknowledged that sharing research data in a timely and
open manner is crucial to furthering progress in science and healthcare.
However, due to the diverse nature of different research approaches,
systems, national guidance and field-specific practice, merely making
data available does not always maximise the utility to the global
community.</p>
<p>With these issues in mind, a group of researchers hosted a
stakeholder workshop with experts from academia and the private sector
in 2014 to discuss how to address the challenges. The workshop generated
a draft set of principles guiding the sharing of data. They agreed that
all research output should be <strong>Findable, Accessible,
Interoperable and Reusable (FAIR)</strong>. These <a
href="https://www.nature.com/articles/sdata201618#Sec6"><strong>FAIR</strong>
guiding principles</a> were subsequently expanded upon as below.</p>
<p>To be <strong>Findable</strong>:<br />
F1. (meta)data are assigned a globally unique and persistent
identifier.</p>
<p>F2. data are described with rich metadata (defined by R1 below)</p>
<p>F3. metadata clearly and explicitly includes the identifier of the
data it describes.</p>
<p>F4. (meta)data are registered or indexed in a searchable
resource.</p>
<p>To be <strong>Accessible</strong>:</p>
<p>A1. (meta)data are retrievable by their identifier using a
standardized communications protocol.</p>
<p>A1.1 the protocol is open, free, and universally implementable.</p>
<p>A1.2 the protocol allows for an authentication and authorization
procedure, where necessary.</p>
<p>A2. metadata are accessible, even when the data are no longer
available.</p>
<p>To be <strong>Interoperable</strong>:</p>
<p>I1. (meta)data use a formal, accessible, shared and broadly
applicable language for knowledge representation.</p>
<p>I2. (meta)data use vocabularies that follow FAIR principles.</p>
<p>I3. (meta)data include qualified references to other (meta)data.</p>
<p>To be <strong>Reusable</strong>:</p>
<p>R1. meta(data) are richly described with a plurality of accurate and
relevant attributes.</p>
<p>R1.1. (meta)data are released with a clear and accessible data usage
license.</p>
<p>R1.2. (meta)data are associated with detailed provenance.</p>
<p>R1.3. (meta)data meet domain-relevant community standards.</p>
<p>These principles were published in 2016 and have been implemented
widely across many fields since then. During the current pandemic, a
number of groups have sought to consider how to implement these
principles with the additional fast-paced pressure of an ongoing
outbreak.</p>
<p>The <a href="https://www.nature.com/articles/s41431-020-0635-7">Virus
Outbreak Data Network (<strong>VODAN</strong>)</a> established a project
to develop an international data network infrastructure supporting
evidence-based responses to the pandemic. One of the projects associated
with <strong>VODAN</strong> has shown that this approach can be <a
href="https://doi.org/10.1186/s13326-022-00263-7">implemented in a
hospital setting</a> to allow rapid linking and sharing of relevant
patient information in a machine-readable format. A further project
highlighted the limited genomic data emerging from Africa underpinned by
concerns around data ownership and availability of health data at the
point of care. A system was developed to provide this data at the point
of care as well as aggregated for global analysis and has been <a
href="https://onlinelibrary.wiley.com/doi/full/10.1002/ggn2.10050">deployed
in a number of African countries</a>.</p>
<p>As well as the <strong>FAIR</strong> principles for data sharing, The
Research Data Alliance International Indigenous Data Sovereignty
Interest Group created a set of principles concerning indigenous data
governance. These principles intend to maximise the benefits of data and
research on individual communities, whilst promoting the outward sharing
of data for global good in a manner that respects previous barriers to
sharing and misuse of data. These principles state that there should be
<strong>Collective benefit, Authority to control, Responsibility and
Ethics <a
href="https://www.nature.com/articles/s41597-021-00892-0#Fig2">(CARE**)</a>.
There have been subsequent efforts to guide the use of both
</strong>FAIR** and <strong>CARE</strong> guidance in data usage (Table
1).</p>
<p><em>Table 1 - Implementation of the CARE Principles across the data
lifecycle</em></p>
<table>
<colgroup>
<col width="25%" />
<col width="25%" />
<col width="25%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Practice CARE in data collection</th>
<th>Engage CARE in data stewardship</th>
<th>Implement CARE in data community</th>
<th>Use FAIR and CARE in data applications</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Define cultural data; record provenance in metadata</td>
<td>Use appropriate governance models; make data FAIR</td>
<td>Indigenous ethics inform access; use tools for transparency,
integrity and provenance</td>
<td>Fairness, accountability, transparency; access equity</td>
</tr>
</tbody>
</table>
<p><em>Source: <a
href="https://doi.org/10.1038/s41597-021-00892-0">Scientific
Data</a></em></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>

<!DOCTYPE html>
<html>
<head>
<style>
* {
  box-sizing: border-box;
}

.img-container-left {
  float: left;
  width: 33.33%;
  padding: 15px;
}

.img-container-right {
  float: right;
  width: 33.33%;
  padding: 15px;
}

.clearfix::after {
  content: "";
  clear: both;
  display: table;
}
</style>
</head>
<body>

<hr>
<div class="clearfix">
  <div class="img-container-left">
    <img src="assets/img/COG-Train_logo.png" alt="COG-Train" 
style="width:200%">
  </div>
  <div class="img-container-right">
    <img src="assets/img/WCS_Logo-Primary_Black.jpg" alt="Wellcome 
Connecting Science" style="width:200%">
  </div>
</div>

</body>
</html>





</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
