<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>practical_guide_week3</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="assets/styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">COG-Train_Resources</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-file-lines"></span>
     
    Course Materials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">Virtual Courses</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Viral Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="vbioinfasia.html">VBioinf2022 Asia</a>
        </li>
        <li>
          <a href="vbioinflac.html">VBioinf2022 LAC</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">MOOCs</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Power of Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="power_of_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="power_of_genomics_set1.html">Week 1</a>
        </li>
        <li>
          <a href="power_of_genomics_set2.html">Week 2</a>
        </li>
        <li>
          <a href="power_of_genomics_set3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Swab to Server</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="swab_to_server_home.html">Home</a>
        </li>
        <li>
          <a href="swab_to_server_week1.html">Week 1</a>
        </li>
        <li>
          <a href="swab_to_server_week2.html">Week 2</a>
        </li>
        <li>
          <a href="swab_to_server_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Making Sense</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="making_sense_home.html">Home</a>
        </li>
        <li>
          <a href="making_sense_week1.html">Week 1</a>
        </li>
        <li>
          <a href="making_sense_week2.html">Week 2</a>
        </li>
        <li>
          <a href="making_sense_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Practical Guide</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="practical_guide_home.html">Home</a>
        </li>
        <li>
          <a href="practical_guide_week1.html">Week 1</a>
        </li>
        <li>
          <a href="practical_guide_week2.html">Week 2</a>
        </li>
        <li>
          <a href="practical_guide_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Pathogen Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="pathogen_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="pathogen_genomics_week1.html">Week 1</a>
        </li>
        <li>
          <a href="pathogen_genomics_week2.html">Week 2</a>
        </li>
        <li>
          <a href="pathogen_genomics_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Hybrid Course</li>
    <li>
      <a href="scvo2b4b.html">SARS-CoV-2 B4B</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">In Person Courses</li>
    <li>
      <a href="ECPG.html">Pathogen Genomics Capacity</a>
    </li>
    <li>
      <a href="IGM_B4B.html">Bioinformática Decodificada</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Webinars</li>
    <li>
      <a href="Community.html">COG-Train Community</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-book"></span>
     
    Team
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cogteam.html">COG-Train Team</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/WCSCourses/COG-Train_Resources">
    <span class="fa fa-brands fa-github"></span>
     
    Github
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="sequencing-analysis-and-interpretation" class="section level1">
<h1>SEQUENCING ANALYSIS AND INTERPRETATION</h1>
<p><img src="images/OC4_3-3_illustrative.png" /><br />
<sub><span class="citation">@Canva</span></sub></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="what-are-sequencing-metrics" class="section level1">
<h1>What are sequencing metrics</h1>
<!-- 3-2 -->
<p><strong>Illumina Sequencing run metrics</strong></p>
<p>Run performance for Illumina sequencing can be monitored using the
Illumina native programmes: Sequencing Analysis Viewer (SAV) and
BaseSpace. Next, we will discuss the key metrics.</p>
<p><em>Yield (Gb)</em></p>
<p>This shows the number of bases generated in the run (Figure 1).
Different sequencers and runs generate different amounts of data over
variable amounts of time.</p>
<div class="float">
<img src="images/OC4_3-2_fig1.png"
alt="Summary of sequencing metrics from BaseSpace: 1. Yield (Gb); 2. % ≥ Q30; 3. % Aligned and % error rate; 4. Cluster Density (K/mm2); 5. %PF" />
<div class="figcaption">Summary of sequencing metrics from BaseSpace: 1.
Yield (Gb); 2. % ≥ Q30; 3. % Aligned and % error rate; 4. Cluster
Density (K/mm2); 5. %PF</div>
</div>
<p><a href="images/OC4_3-2_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-2_Alt-text.pdf">Download Figure 1 to 6 alt-text
here</a></p>
<p><em>% ≥ Q30</em></p>
<p>This is the percentage of bases with a quality score of 30 or higher.
It is evaluated using a Q Score distribution chart that will indicate
the % of reads with Q Score ≥ 30</p>
<ul>
<li>Q30 is the probability of incorrect base calling of 1 in 1000 (99.9%
accuracy)</li>
<li>lllumina runs usually generate &gt; 70-80% Q30 data</li>
<li>The Q30 reported is an average across the whole read length</li>
<li>The Q30 decreases towards the end of the read</li>
</ul>
<p><em>Percent Aligned and Error Rate</em></p>
<p>Illumina recommends that all runs are spiked with their PhiX control;
these metrics are only calculated if PhiX is spiked in.</p>
<ul>
<li>The PhiX library provides a quality control for cluster generation,
sequencing, and alignment.</li>
<li>% Aligned is the percentage of clusters in which the first 25 cycles
align to the PhiX reference genome.</li>
<li>Error rate is the rate of mismatches between sequencing data and the
PhiX reference genome (usually below 1%). If this was not used then
%&gt;=Q30 is your best tool to check base quality.</li>
</ul>
<p><em>Cluster Density (K/mm2)</em></p>
<p>This shows the density of clusters on the flow cell and is an
important metric to evaluate the quality of the data (Figure 1). When
looking at these data you should also consider the <strong>Percentage
Clusters Passing Filter (%PF)</strong>, which is an internal quality
filtering procedure. Together these metrics can help you determine if
there are problems with your loading concentration.</p>
<p>Cluster density and %PF are inversely related: typically if there is
a high cluster density the %PF will be lower and vice versa:</p>
<ul>
<li>Lower %PF may lead to losing coverage on some samples within the
run</li>
<li>If the samples are not inversely related (for example cluster
density is within range but %PF is low) consider an instrument or
reagent issue</li>
</ul>
<p>High cluster density: overloading of the library may cause merging of
clusters which increases the cluster density. This may lead to:</p>
<ul>
<li>Poor template generation, which then causes a decrease in the
percentage passing filter (%PF)</li>
<li>Low Q30 scores</li>
<li>Complete run failure</li>
</ul>
<p>Low cluster density can be due to:</p>
<ul>
<li>Underloading of the library</li>
<li>Library preparation issues (for example, poor denaturation,
etc.)</li>
</ul>
<p>Both high and low cluster densities will have a negative impact on
data output.</p>
<p><strong>Oxford Nanopore sequencing metrics</strong></p>
<p>MinKnow provides real-time information about the sequencing run. It
is important to consider flow cell health, pore occupancy and read
length. If live base-calling is active, quality score plots and
information on the number of reads per barcode can also be followed. A
report is then generated at the end of the run.</p>
<p><em>Flow cell health and pore occupancy</em> The flow cell health
graph provides a summary of the status of all the pores in a flow cell.
A colour code is used to indicate different pore statuses (Figure
2).</p>
<div class="float">
<img src="images/OC4_3-2_fig3.jpeg"
alt="Table showing colour-coding of different flow cell pore occupancy statuses" />
<div class="figcaption">Table showing colour-coding of different flow
cell pore occupancy statuses</div>
</div>
<p><a href="images/OC4_3-2_fig3.jpeg">Click to enlarge the image</a></p>
<p>Pore occupancy information is provided in real-time by the channel
states panel (Figure 3). A high proportion of active sequencing channels
is an indicator of good library preparation.</p>
<div class="float">
<img src="images/OC4_3-2_Fig4.jpeg"
alt="Channel states panel: real-time pore occupancy status of one sequencing run on MinKnow software" />
<div class="figcaption">Channel states panel: real-time pore occupancy
status of one sequencing run on MinKnow software</div>
</div>
<p><a href="images/OC4_3-2_Fig4.jpeg">Click to enlarge the image</a></p>
<p>The pore activity graph summarises the channel state of a run over
time (Figure 4). In a good run, the proportion of active pores decreases
slowly as the run progresses.</p>
<div class="float">
<img src="images/OC4_3-2_fig5.jpeg"
alt="Pore activity: Graph summarising pore activity over time on MinKnow software" />
<div class="figcaption">Pore activity: Graph summarising pore activity
over time on MinKnow software</div>
</div>
<p><a href="images/OC4_3-2_fig5.jpeg">Click to enlarge the image</a></p>
<p><em>Read length histograms and cumulative plots</em></p>
<p>Read length histograms provide cumulative information on the read
length (Figure 5). This information is presented either as number of
reads, or, estimated bases vs read length.</p>
<div class="float">
<img src="images/OC4_3-2_fig6.jpeg"
alt="Read length histogram: Graph showing estimated read length vs estimated bases on MinKnow software" />
<div class="figcaption">Read length histogram: Graph showing estimated
read length vs estimated bases on MinKnow software</div>
</div>
<p><a href="images/OC4_3-2_fig6.jpeg">Click to enlarge the image</a></p>
<p>Cumulative output graphs show the total number of reads and if they
have passed or failed the quality filter (Figure 6). If live
base-calling is not active, only the total number of reads will be
displayed. If live base-calling is active, cumulative median or modal
quality score plots of passed reads are available.</p>
<div class="float">
<img src="images/OC4_3-2_fig7.jpeg"
alt="Cumulative output: Graph showing the total number of reads and whether they have passed or failed the quality filter on MinKnow software" />
<div class="figcaption">Cumulative output: Graph showing the total
number of reads and whether they have passed or failed the quality
filter on MinKnow software</div>
</div>
<p><a href="images/OC4_3-2_fig7.jpeg">Click to enlarge the image</a></p>
<p><em>Barcode stats</em></p>
<p>If multiple samples are being sequenced, the barcode read counts
graph displays the number of reads that have been basecalled and have
passed the quality filter for each barcode.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="accessing-and-using-sequencing-data" class="section level1">
<h1>Accessing and using sequencing data</h1>
<!-- 3-3 -->
<p><strong>Sequencing platforms</strong></p>
<p>There are several different sequencing platforms commercially
available. Although we focus here on those developed by <a
href="https://www.illumina.com/">Illumina</a> and <a
href="https://nanoporetech.com/">Oxford Nanopore Technologies (ONT)</a>,
other available platforms include PacBio by <a
href="https://www.pacb.com/">Pacific Biosciences</a> and Ion Torrent by
<a
href="https://www.thermofisher.com/uk/en/home/brands/ion-torrent.html">Thermo
Fisher Scientific</a> In high-throughput sequencing (HTS), also known as
next-generation sequencing (NGS), sequencers create millions of reads. A
single read is a computational DNA sequence created from the sequencing
of one piece of DNA in the library that is being sequenced. A sequencing
library is a pool of DNA fragments with sequencing adapters and barcodes
attached. The library pool can be composed of DNA from multiple samples
each with its own unique barcode.</p>
<p>It should be noted that whilst Illumina sequencers only sequence DNA
molecules, RNA transcriptomic sequence analyses can still be conducted
by performing a reverse transcription of the sample RNA to DNA before
library preparation, whilst Oxford Nanopore sequencers are capable of
sequencing either RNA or DNA molecules directly. In the descriptions
below, we use DNA as the molecule being sequenced.</p>
<p><strong>Sequence data formats and conversion</strong></p>
<p>Sequencers create millions of reads, which need gigabytes (GB) of
computational storage.</p>
<p>The standard format used to computationally store reads is the FASTQ
format (Figure 1), which is a text-based format for storing nucleotide
sequences along with their corresponding quality scores. Quality scores
are included for each sequenced base which represents the sequencer’s
probability estimate that the base was sequenced correctly.</p>
<div class="float">
<img src="images/OC4_3-3_fig1.png"
alt="Example FASTQ read. Four text lines are used to represent each read. The first line is the unique name of the read, the second line is the nucleotide sequence, the third link is the ‘break’ line, and the fourth line is the quality scores (one for each base sequenced). A table of quality score characters and their corresponding error probabilities can be found at Usearch website" />
<div class="figcaption">Example FASTQ read. Four text lines are used to
represent each read. The first line is the unique name of the read, the
second line is the nucleotide sequence, the third link is the ‘break’
line, and the fourth line is the quality scores (one for each base
sequenced). A table of quality score characters and their corresponding
error probabilities can be found at <a
href="https://www.drive5.com/usearch/manual/quality_score.html">Usearch
website</a></div>
</div>
<p><a href="images/OC4_3-3_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-3_Alt-text.pdf">Download Figure 1 and alt-text
here</a></p>
<p>Illumina machines output their raw data in binary base call (BCL)
format. Some Illumina sequencers (MiniSeq, MiSeq and NextSeq) provide an
option to automatically convert the reads from BCL to the FASTQ format.
Whilst other Illumina sequencers (NovaSeq) require separate conversions.
This can be accomplished manually by using Illumina’s <a
href="https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html">BCL
Convert</a> software (a replacement of the older <a
href="https://emea.support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html">bcl2fastq</a>
software). Alternatively, Illumina has a cloud-based data management and
bioinformatics system called BaseSpace (a subscription may be required).
Sequencing data is uploaded to BaseSpace which enables real-time
monitoring of sequence runs and conversion from BCL to FASTQ formats in
the cloud, amongst a host of other functionalities.</p>
<p>All Oxford Nanopore sequencing devices (MinION, GridION, PromethION,
etc), use nanopores corresponding to an electrode connected to a channel
and sensor chip. The sensor measures the change in the electric current
that flows through the nanopore when DNA or RNA molecules pass through.
This electric signal data produced from a nanopore read is referred to
as a ‘squiggle’ and is stored computationally using the FAST5 format
(which is based on the HDF5 format). FAST5 squiggles (electrical reads)
can then be converted into the standard FASTQ sequence format by base
calling. This can be accomplished using software such as Guppy, which
can be run manually post-run or run ‘live’ during the run; live base
calling works fastest when the computer connected to the nanopore
sequencer has a Graphical Processing Unit (GPU). Alternatively, Oxford
Nanopore provides a cloud-based data analysis platform called <a
href="https://epi2me.nanoporetech.com/">EPI2ME</a> (subscription may be
required) where FAST5 data can be uploaded during the sequencing run and
base calling can be done live, amongst a host of other
functionalities.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-generate-a-consensus-sequence" class="section level1">
<h1>How to generate a consensus sequence</h1>
<!-- 3-4 -->
<p><strong>Demultiplexing</strong></p>
<p>As sequencers create large numbers of reads, the library being
sequenced can consist of multiple samples that are pooled together for
sequencing. To determine which reads have come from which sample in the
pool, unique barcode sequences (one barcode for each sample being
sequenced) are added during the library preparation step. When the
library is sequenced, each read will contain (at its ends) the unique
barcodes that are used to label the samples.</p>
<p>Demultiplexing is the process of sorting sequencing reads based on
the unique barcode sequences into different files, with each file
representing one of the samples being sequenced. Both Illumina (through
BCL convert or BaseSpace) and Oxford Nanopore Technology (ONT) (through
guppy or EPI2ME) platforms can demultiplex automatically. However, you
must provide the software with a list of sample names and the
corresponding barcode sequences used for each sample. It is important to
note that demultiplexing is not 100% accurate. For example, sometimes
multiple sequencing errors occur within the barcode of a read leading to
a read being undetermined or assigned to the wrong sample
(misassigned).</p>
<p><strong>Data download</strong></p>
<p>Although there are some cloud-based systems (such as BaseSpace and
EPI2ME) which offer a number of options to analyse your data, it is
typical to transfer sequence reads onto your local computer or server
for analysis. Read files can be large and are therefore usually
compressed to reduce data storage needs and speed up data transfer
processes. If you are using BaseSpace or EPI2ME then you can simply
download your FASTQ reads through their corresponding web interfaces
onto your local machine.</p>
<p>Alternatively, you can utilise file transfer software that can
transfer data from one machine (e.g. a sequencer) to another machine
(e.g. a computer) over the internet. There are several free tools that
can be used to do this, such as <a
href="https://cyberduck.io/">Cyberduck</a> and <a
href="https://filezilla-project.org/">FileZilla</a>, which have easy to
use graphical interfaces. It is standard to use the Secure File Transfer
Protocol (SFTP) option within these tools as this encrypts the data
while it is being transferred. Security is especially important when
dealing with human sequence data.</p>
<p>It is important to note that your raw sequence data is very valuable,
and it is therefore very important to keep adequate backups of your data
in case of any data loss disasters.</p>
<p><strong>Data quality control</strong></p>
<p>We now have our sequence reads demultiplexed into different sample
FASTQ files. The first step in any analysis is quality control (garbage
in = garbage out) which typically involves evaluating the number and
quality of the reads. Common QC steps are removing any leftover adapter
sequences from the ends of reads, removing poor-quality reads, and
trimming poor-quality ends from reads (primarily for Illumina reads, as
sequenced base quality typically deteriorates along the read length).
The overall aim is to remove potentially erroneous reads (or erroneous
parts of reads) from the data, which will improve downstream analyses
such as reference alignments, consensus and variant calling.</p>
<p><a
href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FASTQC</a>
is one of the most popular tools used for visual inspection of read
quality, as is <a
href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">Trim
Galore!</a> for adapter and quality trimming of Illumina reads; although
there are many alternatives such as <a
href="http://prinseq.sourceforge.net/">PRINSEQ</a>, <a
href="http://www.usadellab.org/cms/index.php?page=trimmomatic">Trimmomatic</a>,
and <a href="https://github.com/najoshi/sickle">Sickle</a>, to name a
few.</p>
<p>For ONT data, tools for inspection and summary statistics of reads
include <a href="https://github.com/a-slide/pycoQC">PycoQC</a>, <a
href="https://github.com/wdecoster/nanostat">NanoStat</a>, <a
href="https://github.com/davidebolo1993/NanoR">NanoR</a>, and <a
href="https://github.com/esteinig/nanoq">NanoQ</a>.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="user-friendly-tools-for-quality-control"
class="section level1">
<h1>User-friendly tools for quality control</h1>
<!-- 3-5 -->
<p><strong>Reference alignment and consensus sequence
generation</strong></p>
<p>There are many analyses that can be done with a sample’s FASTQ reads,
but here we will focus on reference alignment. This is typically used
when you have a good idea of what biological organism is in the sample
and a reference sequence for that organism is used to align all your
sequence reads to.</p>
<p>For example, for a SARS-CoV-2 sample, the SARS-CoV-2 genome sequence
from the start of the pandemic (Wuhan-Hu-1) is typically used as a
reference sequence. Each of the sequence reads is then taken in turn and
aligned to this reference sequence to determine the exact section of the
reference genome that the read has come from. It is important to note
that it’s likely that not all the reads will be able to align to the
reference sequence e.g. some reads may well originate from the host
(e.g. human).</p>
<p>There are many tools available to perform reference alignment of
reads, a few of the most commonly used aligners are <a
href="http://bio-bwa.sourceforge.net/">BWA</a>, <a
href="http://bowtie-bio.sourceforge.net/bowtie2/">BOWTIE2</a>, and <a
href="https://doi.org/10.1093/bioinformatics/bty191">Minimap2</a> (which
works with both Illumina and Oxford Nanopore (ONT) data); although there
are many other aligners available. These tools are all optimised to
rapidly align millions of reads to a reference sequence. Each read is
aligned to the reference sequence separately, and the alignment results
(reference genome position, number of mutations, insertions and
deletions) are stored in a results file that is typically outputted in
the standard Sequence Alignment Map (SAM) format. SAM files are
typically converted into Binary Alignment Map (BAM) format files which
are binary compressed versions and therefore use fewer data storage
space and are faster to work with for downstream analyses.</p>
<p>SAM/BAM files store the alignment results of every read to the
reference genome and enable the creation of a “consensus” sequence for
the sample. A consensus sequence can be defined as the most frequent
base at each genome position. It is important that insertions and
deletions (indels) with respect to the reference sequence are also
considered, not just the frequency of the DNA nucleotides A, C, G and T.
Consensus callers work by essentially moving along the reference
alignment, considering each genome position in turn, evaluating all the
reads that are aligned at the position, and determining the most
frequent nucleotide (or indel). The consensus at each genome position is
then combined to give a consensus genome sequence for the sample (Figure
2).</p>
<div class="float">
<img src="images/OC4_3-5_fig1.png"
alt="Example read alignment. Each read has been independently aligned to the reference sequence. Mutations within each read and with respect to the reference sequence are highlighted in red. Based on all the read alignments combined we can calculate a consensus sequence – the most frequent base observed at each genome position. As the majority of reads contain the two highlighted mutations, these are incorporated in the consensus sequence. This is an example with a short genome and low coverage with only between one and three reads covering each position" />
<div class="figcaption">Example read alignment. Each read has been
independently aligned to the reference sequence. Mutations within each
read and with respect to the reference sequence are highlighted in red.
Based on all the read alignments combined we can calculate a consensus
sequence – the most frequent base observed at each genome position. As
the majority of reads contain the two highlighted mutations, these are
incorporated in the consensus sequence. This is an example with a short
genome and low coverage with only between one and three reads covering
each position</div>
</div>
<p><a href="images/OC4_3-5_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-5_Alt-text.pdf">Download Figure 2 alt-text
here</a></p>
<p>There are several tools available to call a consensus sequence from a
SAM/BAM file. Perhaps the most commonly used tools during the SARS-CoV-2
pandemic are <a href="https://github.com/andersen-lab/ivar">iVar</a>
(Illumina data), and the ARTIC bioinformatics protocol (ONT data). One
important aspect of consensus calling is the minimum depth needed to
reliably call the consensus base at a given genome position. Depth (or
coverage) is the number of reads that are aligned at a position. Due to
sequencing errors (as well as errors potentially caused during reverse
transcription and/or PCR amplification), sufficient data is needed to be
sure the consensus base is real and not just a random error observed
when coverage is low. For SARS-CoV-2 samples, a coverage threshold of 10
for Illumina data and 20 for ONT data has been commonly used.</p>
<p>Once you have a consensus sequence, you can evaluate the mutations it
has with respect to the reference sequence and investigate their
functional effect (i.e. where the mutation is, and whether it results in
an amino acid change), the sequence can be BLASTed online to identify
similar sequences, and added to phylogenetic trees to investigate
relationships.</p>
<p>This has been a relatively brief overview of assembly and consensus
sequence generation, some additional points to consider are:</p>
<ul>
<li>When sharing any data, you want to avoid sharing any human patient
reads. One way to accomplish this is by removing any reads that do not
match to your target reference genome.</li>
<li>SARS-CoV-2 genome sequencing has typically been accomplished using
ARTIC amplicon sequencing. Amplicons will have primer sequences
incorporated at the ends of their reads which need to be removed as they
do not come from the sample DNA. This has typically been accomplished
post reference alignment (but pre-consensus calling) using iVar for
Illumina data, and the ARTIC bioinformatics protocol for Oxford Nanopore
data.</li>
<li>Sequencing controls are important to determine the level (and type)
of contamination present in the laboratory. If water is used as a
negative control and you are able to create a SARS-CoV-2 consensus
genome sequence (due to contamination), can you reliably trust the
results of your other samples on the run?</li>
<li>An alternative to reference alignment is de novo assembly, which is
often used when you are not sure what is in your sample or you suspect a
significant difference to your reference sequence. <em>De novo</em>
attempts to piece your individual reads back together, based on shared
sequence overlaps, to re-create any genome sequences in the sample. A
commonly used tool for de novo assembly is SPAdes for Illumina data and
Canu for Oxford Nanopore data.</li>
<li>If you are writing a scientific publication that includes analyses
of sequence reads, it is standard practice to upload your reads (with
any human patient reads removed) into online repositories such as the <a
href="https://www.ncbi.nlm.nih.gov/sra">NCBI Short Read Archive</a> or
<a href="https://www.ebi.ac.uk/ena/browser">European Nucleotide
Archive</a>. Sequence reads stored on these repositories are publicly
available for download onto your computer either via their websites or
using command line tools such as the <a
href="https://hpc.nih.gov/apps/sratoolkit.html">SRA-Toolkit</a>.</li>
</ul>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="why-high-quality-data-is-important" class="section level1">
<h1>Why high-quality data is important</h1>
<!-- 3-6 -->
<p>A growing number of biological topics may now be understood, thanks
to high-throughput sequencing methods. These technologies have continued
to advance, and, most recently, long-read sequencing has expanded its
applications in genomics, transcriptomics, and metagenomics by
surpassing earlier accuracy constraints. This means that generating a
huge amount of data requires curation before any downstream analysis.
Next-generation sequencing’s most popular raw data format is identified
by the FASTQ file extension. Sequence data and a quality score for each
site are included in the data format created by sequencing
platforms.</p>
<p><strong>FASTQ Format</strong></p>
<p>The FASTQ file contains four lines of information (Figure 2):</p>
<ol style="list-style-type: decimal">
<li>The header line begins with @ and has information about the unique
instrument name, run ID, flow cell ID, flowcell lane, tile number, X and
Y coordinates of clusters in the tile, paired(1) or mate-pair(2), status
about the read is filtered or not (Y – filtered, N-not), control sample
status (0 when none), and index sequence<br />
</li>
<li>A, T, G and C of the read<br />
</li>
<li>A line with (+) sign<br />
</li>
<li>Quality score of each base in ASCII format</li>
</ol>
<p><strong>Quality control</strong></p>
<p>Quality control generally involves calculating the number of reads,
GC content to identify overrepresented sequences, discarding reads with
Ns (uncalled bases), removing adaptor sequences, and removing or
trimming low-quality bases. These are necessary to lessen the likelihood
of bias in variant calling and/or new assembly before any downstream
process (Figure 3).</p>
<div class="float">
<img src="images/OC4_3-6_fig1.png"
alt="Flowchart showing the steps involved in the pre-processing Quality Control step" />
<div class="figcaption">Flowchart showing the steps involved in the
pre-processing Quality Control step</div>
</div>
<p><a href="images/OC4_3-6_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-6_Alt-text.pdf">Download Figure 3-5 alt-text
here</a></p>
<p>There are several tools which perform quality checks for the given
reads such as FastQC, Scythe and Sickle, but, FastQC is the most
commonly used program. This program performs various tests and produces
basic statistics indicating pass (green tick), warning (yellow) and
failed (red). It produces, per tile, Sequence Quality in which the
colour of the tiles indicates the read quality which can be seen in
cells. Per-base Sequence Content provides the percentage content of A,
T, G and C base within a read. Per base, N content will provide Ns
present in a given read. Reads with Ns are normally discarded. Sequence
Duplication Levels show the level of duplication, a high level shows
bias in the enrichment step. Adapter content will show the presence or
absence of adapters.</p>
<p><strong>Quality Scores</strong></p>
<p>Sequencing quality scores measure the probability that a base is
called incorrectly. With sequencing by synthesis (SBS) technology, each
base in a read is assigned a quality score by a phred-like algorithm,
similar to that originally developed for Sanger sequencing
experiments.</p>
<p><em>Q Score Definition</em></p>
<p>The sequencing quality score of a given base, Q, is defined by the
following equation:</p>
<pre><code>Q = -10log10(e)</code></pre>
<p>where e is the estimated probability of the base call being
wrong.</p>
<p>Higher Q scores indicate a smaller probability of error. Lower Q
scores can result in a significant portion of the reads being
unusable.</p>
<p>To identify low base quality <a
href="https://doi.org/10.1101/gr.8.3.186">Based on the Phred
scale(Q)</a>, all quality scores are determined. Each base call has a
corresponding base call quality that calculates the likelihood that the
base call is inaccurate.</p>
<p><em>Table 1 - Relationship between sequencing quality score and base
call accuracy</em></p>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th>Quality score</th>
<th>Probability of incorrect base call</th>
<th>Inferred base call accuracy</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0 (Q10)</td>
<td>1 in 10</td>
<td>90%</td>
</tr>
<tr class="even">
<td>20 (Q20)</td>
<td>1 in 100</td>
<td>99%</td>
</tr>
<tr class="odd">
<td>30 (Q30)</td>
<td>1 in 1000</td>
<td>99.9%</td>
</tr>
</tbody>
</table>
<p>The quality scores may become erroneous due to extreme GC biases,
specific patterns, or homopolymers. To guarantee that assembly or
variant calls are accurate, accurate base characteristics are a crucial
component. It is assumed that Illumina data with less than Q20 is not
valuable data and should be eliminated as a general rule for Illumina
short reads. In case of long reads from Pacbio or Nanopore, it is around
Q10, as the error rate is high in these technologies.</p>
<p><strong>MultiQC</strong></p>
<p>When analysing multiple samples, checking QC for each one
individually is time-consuming. MultiQC offers a solution for this
stage. <a href="https://multiqc.info/">MultiQC</a> aggregates reports
from many experiments, including FastQC, and provides a single output
(Figure 4 and 5).</p>
<div class="float">
<img src="images/OC4_3-6_fig2.png"
alt="Quality score plots. Per base sequence quality, which plots the Q-score of the raw sequence, reads as a box-plot for each cycle. Higher is always better, and a characteristic quality decay is seen in most runs" />
<div class="figcaption">Quality score plots. Per base sequence quality,
which plots the Q-score of the raw sequence, reads as a box-plot for
each cycle. Higher is always better, and a characteristic quality decay
is seen in most runs</div>
</div>
<p><a href="images/OC4_3-6_fig2.png">Click to enlarge the image</a></p>
<div class="float">
<img src="images/OC4_3-6_fig3.png"
alt="Sequence content plot. Per base sequence content, which plots the proportion of each base at each cycle. In a random fragment library from a “normal” genome you would expect to see all four bases equally represented. Deviation from normal base content can indicate issues with library quality, but, equally, some genomes are very GC biased and some NGS applications also introduce a strong GC bias, e.g. Bis-seq" />
<div class="figcaption">Sequence content plot. Per base sequence
content, which plots the proportion of each base at each cycle. In a
random fragment library from a “normal” genome you would expect to see
all four bases equally represented. Deviation from normal base content
can indicate issues with library quality, but, equally, some genomes are
very GC biased and some NGS applications also introduce a strong GC
bias, e.g. Bis-seq</div>
</div>
<p><a href="images/OC4_3-6_fig3.png">Click to enlarge the image</a></p>
<p><strong>Contamination with other species</strong></p>
<p>Sampling and DNA extraction errors may cause contaminants in the
reads. There are now several tools that swiftly search reads and assign
them to specific species or taxonomic groups. Tools like Kraken, Kaiju
and Centrifuge are used to identify these contaminants. Kraken analysis
provided the top species for each isolate, along with relatedness
percentages for any other species.</p>
<p><strong>Further reading</strong></p>
<p><a
href="https://www.illumina.com/documents/products/technotes/technote_understanding_quality_scores.pdf">Understanding
Illumina Quality Scores</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="what-are-clades-lineages-and-variants" class="section level1">
<h1>What are clades, lineages and variants?</h1>
<!-- 3-7 -->
<p>Genomics helps us to track the spread of an infectious disease
outbreak and to identify significant changes in the genome of a
pathogen, including the <a
href="https://doi.org/10.1093/ve/veab064">identification of new variants
or lineage</a>.</p>
<p>Using web interfaces such as <a
href="https://cov-lineages.org/resources/pangolin.html">Pangolin</a> and
<a href="https://nextstrain.org/ncov/open/global/6m">Nextstrain</a> to
perform phylogenetic analysis has helped with genomic surveillance,
which has been extremely important during the pandemic period. Pangolin
stands for Phylogenetic Assignment of Named Global Outbreak Lineages.
This is a tool that allows a user to classify a coronavirus based on its
genetic relatedness to others. We use it to differentiate one
transmission chain from another (e.g. local nosocomial versus school
outbreaks). However, in order to do this, we needed to create categories
into which we could classify the diversity. This was done by <a
href="https://doi.org/10.1038/s41564-020-0770-5">dividing the groups
into clades and lineages</a>.</p>
<p><strong>So, what are clades, lineages and variants?</strong></p>
<p>A clade is a very broad way of grouping SARS-CoV-2 isolates. Hence,
it gives us a sense of the diversity patterns over the years. The
samples also need to have a minimum size and persistence.</p>
<p>A lineage is a more specific grouping of SARS-CoV-2 isolates. The
lineage rules for SARS-CoV-2 were proposed by <a
href="https://doi.org/10.1038/s41564-020-0770-5">Rambaut and
colleagues</a> to help us describe the diversity we are observing while
the pandemic is happening and attempts to describe a significant
epidemiological event. For example, a new lineage may be assigned if
that specific group shows an increase in transmission compared to
another group. There are no criteria for minimum size and persistence.
Hence a lineage may contain a small number of isolates or a very large
number. For example, there are over <a
href="https://doi.org/10.1093/ve/veab064">1,000 Pango lineages</a>,
compared to less than 20 clades.</p>
<p>Lastly, a <a href="https://doi.org/10.1016/j.yamp.2021.06.006">virus
variant</a> has mutations that have a biological significance (for
example, it is associated with a virulence factor). The most significant
variants are currently called <a
href="https://doi.org/10.1128/jvi.02077-21">Alpha, Beta, Gamma, Delta
and Omicron</a>.</p>
<p>Below are two images from <a
href="https://nextstrain.org/ncov/open/global/6m">Nextstrain</a> showing
phylogenetic analysis of different variants of SARS-CoV-2 (Figure 6 and
7). This platform is updated often, so the results may look a bit
different by the time you view it.</p>
<div class="float">
<img src="images/OC4_3-7_fig1.jpeg"
alt="Example results from Nextstrain part 1" />
<div class="figcaption">Example results from Nextstrain part 1</div>
</div>
<p><a href="images/OC4_3-7_fig1.jpeg">Click here to enlarge the
image</a></p>
<p><a href="images/OC4_3-7_Alt-text.pdf">Download Figure 6 and 7
alt-text here</a></p>
<div class="float">
<img src="images/OC4_3-7_fig2.jpeg"
alt="Example results from Nextstrain part 2" />
<div class="figcaption">Example results from Nextstrain part 2</div>
</div>
<p><a href="images/OC4_3-7_fig2.jpeg">Click here to enlarge the
image</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="sars-cov-2-variants" class="section level1">
<h1>SARS-CoV-2 Variants</h1>
<!-- 3-8 -->
<p>The World Health Organisation (WHO) <a
href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">classifies
variants</a> according to criteria which include the variant makeup and
the spread of the variant virus.</p>
<p>Pango is short for <a
href="https://cov-lineages.org/resources/pangolin.html">Pangolin</a>
classification. Pangolin stands for Phylogenetic Assignment of Named
Global Outbreak Lineages. This is a tool that allows a user to classify
a coronavirus based on its genetic relatedness to others. Genetic
relatedness is rooted in the virus’s genetic lineage (the virus strain
which came before it). Phylogenetics is the study of the genetic
evolutionary lineage of an organism.</p>
<p>Other largely used systems to classify and name SARS-CoV-2 variants
are <a
href="https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming">Nextstrain
clade</a> and <a
href="https://www.gisaid.org/resources/statements-clarifications/clade-and-lineage-nomenclature-aids-in-genomic-epidemiology-of-active-hcov-19-viruses/">GISAID
clade</a></p>
<p><strong>Variants of interest (VOI)</strong></p>
<p>There is no current circulating VOI. To be considered at VOI, a
SARS-CoV-2 variant must meet these two criteria:</p>
<ul>
<li><p>With genetic changes that are predicted or known to affect virus
characteristics such as transmissibility, disease severity, immune
escape, diagnostic or therapeutic escape;</p></li>
<li><p>Identified to cause significant community transmission or
multiple COVID-19 clusters, in multiple countries with increasing
relative prevalence alongside an increasing number of cases over time,
or other apparent epidemiological impacts to suggest an emerging risk to
global public health.</p></li>
</ul>
<p><strong>Variants of concern (VOC)</strong></p>
<p>To be considered a VOC, a VOI must meet one or more of the following
criteria, at a level that has a significant effect on public health:</p>
<ul>
<li><p>Increase in transmissibility or detrimental change in COVID-19
epidemiology; OR</p></li>
<li><p>Increase in virulence or change in clinical disease presentation;
OR</p></li>
<li><p>Decrease in the effectiveness of public health and social
measures or available diagnostics, vaccines or therapeutics.</p></li>
</ul>
<p><em>Table 1 - SARS-CoV-2 variants of concern. In September 2022, only
Omicron is considered a VOC. Alpha, Beta, Gamma and Delta are previously
circulating VOCs. Source: <a
href="https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/">WHO</a></em></p>
<table>
<colgroup>
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th>WHO Label</th>
<th>Pango lineage</th>
<th>GISAID clade</th>
<th>Nextstrain clade</th>
<th>Earliest documented samples</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Omicron</strong></td>
<td><strong>B.1.1.529</strong></td>
<td><strong>GR/484A</strong></td>
<td><strong>21K, 21L, 21M, 22A, 22B, 22C, 22D</strong></td>
<td><strong>Multiple countries Nov-2020</strong></td>
</tr>
<tr class="even">
<td>Alpha</td>
<td>B.1.1.7</td>
<td>GRY</td>
<td>20I (V1)</td>
<td>United Kingdom Sep-2020</td>
</tr>
<tr class="odd">
<td>Beta</td>
<td>B.1.351</td>
<td>GH/501Y.V2</td>
<td>20H (V2)</td>
<td>South Africa May-2020</td>
</tr>
<tr class="even">
<td>Gamma</td>
<td>P1</td>
<td>GR/501Y.V3</td>
<td>20J (V3)</td>
<td>Brazil Nov-2020</td>
</tr>
<tr class="odd">
<td>Delta</td>
<td>B.1.617.2</td>
<td>G/478K.V1</td>
<td>21A, 21I, 21J</td>
<td>India Oct-2020</td>
</tr>
</tbody>
</table>
<p>NB: Omicron lineage includes BA.1, BA.2, BA.3, BA.4, BA.5 and
descendent lineages.</p>
<p><strong>Variants of concern under monitoring (VOC-LUM)</strong></p>
<p>Since its emergency, Omicron has already displaced other
co-circulating SARS-CoV-2 variants. Thus, WHO has added a new category
termed VOC-LUM to identify VOC lineages that may require prioritised
attention and monitoring.</p>
<p>A VOC-LUM is:</p>
<ul>
<li><p>A variant that, according to phylogenetic analysis, belongs to a
currently circulating VOC; AND</p></li>
<li><p>Shows signals of transmission advantage compared to other
circulating VOC; AND</p></li>
<li><p>Has additional amino acid changes that are known or suspected to
confer the observed change in epidemiology and fitness advantage as
compared to other circulating variants.</p></li>
</ul>
<p>A full list of Pango lineages is available on the <a
href="https://cov-lineages.org/lineage_list.html">Pango website</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="interpreting-nextclade-quality-metrics" class="section level1">
<h1>Interpreting Nextclade quality metrics</h1>
<!-- 3-9 -->
<p>Nextclade is a free and open-source tool for viral genome alignment,
mutation calling, clade assignment, quality checks, and phylogenetic
positioning. It can rapidly assess viral genomes by running a series of
simple, but useful, calculations that can be used to inspect and
classify your newly sequenced genomes or as a pre-submission check prior
to depositing them in public databases like NCBI and GISAID. It is
developed by the same team behind <a
href="https://nextstrain.org">Nextstrain</a>.</p>
<p>Nextclade is available through a command line or web-based interface
available at <a href="https://clades.nextstrain.org">Nextclade
website</a>. You can drag and drop a file containing your SC-2 genomes
into your browser window, retrieve sequences from a URL, or paste your
FASTA sequences into the text field. Nextclade will detect differences
between your sequences and a reference sequence <a
href="https://www.ncbi.nlm.nih.gov/nuccore/MN908947.3">(Wuhan-Hu-1/2019)</a>,
use these differences to classify your sequences into clades/lineages,
and flag probable quality issues in your data. All of the analysis takes
place entirely within the browser, with no data ever leaving the
computer.</p>
<p>The analytical pipeline includes the following steps:</p>
<ol style="list-style-type: decimal">
<li>Sequence alignment: Using a custom alignment algorithm, sequences
are aligned to the reference genome.</li>
<li>Translations: Nucleotide sequences are translated into amino acid
sequences.</li>
<li>Mutation calling: Changes in nucleotides and amino acids are
found.</li>
<li>Detection of PCR primer changes in binding sites</li>
<li>Phylogenetic placement: Sequences are placed on a reference tree,
and private mutations are examined.</li>
<li>Assignment of Clades: Clades are derived from the tree’s parent
node.</li>
<li>Quality Control (QC): Metrics for quality control are computed.</li>
</ol>
<p>In this example, we examine 61 SC-2 genomes from Peru that were
recently sequenced using an Illumina MiSeq instrument. When you press
the “run” button, you will immediately see an overview of the results
(Figures 8 and 9).</p>
<div class="float">
<img src="images/OC4_3-9_fig1.png" alt="Nextclade input page" />
<div class="figcaption">Nextclade input page</div>
</div>
<p><a href="images/OC4_3-9_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-9_Alt-text.pdf">Download Figure 8-13 alt-text
here</a></p>
<div class="float">
<img src="images/OC4_3-9_fig2.png" alt="Nextclade run output" />
<div class="figcaption">Nextclade run output</div>
</div>
<p><a href="images/OC4_3-9_fig2.png">Click to enlarge the image</a></p>
<p>Nextclade uses several quality control indicators to quickly identify
problems in your sequences. Bad sequences are red, mediocre ones are
yellow, and good ones are white. By hovering your cursor over a
sequence’s QC entry, you can get the results of the QC metrics (Figure
10).</p>
<div class="float">
<img src="images/OC4_3-9_fig3.png"
alt="Nextclade output: additional information" />
<div class="figcaption">Nextclade output: additional information</div>
</div>
<p><a href="images/OC4_3-9_fig3.png">Click to enlarge the image</a></p>
<p>Nextclade also infers the clade/lineage to which a sequence belongs
and displays the result in the table (Figure 11). Hovering over the
results displays specific information such as the number of mutations,
Ns, gaps, insertions, frameshifts, and so on.</p>
<div class="float">
<img src="images/OC4_3-9_fig4.png"
alt="Nextclade output: nucleotide substitutions" />
<div class="figcaption">Nextclade output: nucleotide substitutions</div>
</div>
<p><a href="images/OC4_3-9_fig4.png">Click to enlarge the image</a></p>
<p>The alignment can be seen at the right of the window, with missing
data shown in grey. You can readily see how missing data segments are
scattered around the genome, whether it’s a few large portions clustered
in one location or many small missing parts.</p>
<p>You can zoom in on a gene by clicking on it at the bottom, or by
selecting it from the dropdown menu at the top. In sequence view, you
can see mutations in a specific gene, which can be shown next to its
codon translation (Figure 12).</p>
<div class="float">
<img src="images/OC4_3-9_fig5.png"
alt="Nextclade output: amino acid translation" />
<div class="figcaption">Nextclade output: amino acid translation</div>
</div>
<p><a href="images/OC4_3-9_fig5.png">Click to enlarge the image</a></p>
<p>When Nextclade is done with its analysis, you can download the
results in different formats by clicking the download icon in the upper
right corner (Figure 13).</p>
<div class="float">
<img src="images/OC4_3-9_fig6.png" alt="Nextclade export window" />
<div class="figcaption">Nextclade export window</div>
</div>
<p><a href="images/OC4_3-9_fig6.png">Click to enlarge the image</a></p>
<p><strong>Further reading</strong></p>
<p><a href="https://doi.org/10.21105/joss.03773">Nextclade: clade
assignment, mutation calling and quality control for viral
genomes</a></p>
<p><a
href="https://chanzuckerberg.zendesk.com/hc/en-us/articles/360052479232-Viewing-samples-in-Nextclade">Nextclade
tutorial at IDseq</a></p>
<p><a
href="https://docs.nextstrain.org/projects/nextclade/en/stable/">Nextstrain
FAQs and documentation</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="lineages-classification-using-pangolin" class="section level1">
<h1>Lineages classification using Pangolin</h1>
<!-- 3-10 -->
<p><a
href="https://cov-lineages.org/resources/pangolin.html">Pangolin</a>
(Phylogenetic Assignment of Named Global Outbreak Lineages) is a piece
of software created by the same team that proposed the <a
href="https://www.nature.com/articles/s41564-020-0770-5">Pango
nomenclature scheme</a>, which we discussed earlier in the course. It’s
easy to use and allows labs to quickly assign the most likely Pango
lineage to newly sequenced data.</p>
<p>Installing is simple for users who are familiar with the UNIX
environment, as it only requires a Python interpreter to run Pangolin
and the Conda system to install dependencies.</p>
<p>Pangolin is also available as a <a
href="https://pangolin.cog-uk.io/">web-based application</a> for less
experienced users. Users can submit SARS-CoV-2 genome sequences in FASTA
format by dragging and dropping or selecting from the browser’s file
browser. The sequences will be analysed using the Pangolin software
after they have been uploaded (versions of the software and the
underlying data are displayed at the bottom of the page).</p>
<p>When the processing is complete, the lineage for each sample, as well
as the bootstrap support values, are displayed. Links to additional
information on the lineage and its visualisation within the <a
href="https://microreact.org/">Microreact</a> platform are also
provided. The results are available as a CSV spreadsheet file download.
Almost three million virus genomes have been uploaded to Pangolin’s web
application to date, enabling SARS-CoV-2 genomic epidemiology and
providing scientists with key information about viral lineage
transmission. Complete instructions for using the Pangolin <a
href="https://pangolin.docs.cog-uk.io/">web application</a> and a
tutorial for the <a
href="https://cov-lineages.org/resources/pangolin/usage.html">command-line
version</a> are available at <a
href="https://cov-lineages.org/index.html">Cov-Lineages website</a>.</p>
<p><strong>Further reading</strong></p>
<p><a href="https://doi.org/10.1038/s41564-020-0770-5">A dynamic
nomenclature proposal for SARS-CoV-2 lineages to assist genomic
epidemiology</a></p>
<p><a href="https://doi.org/10.1093/ve/veab064">Assignment of
epidemiological lineages in an emerging pandemic using the Pangolin
tool</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="introduction-to-phylogenetics-tools-and-models"
class="section level1">
<h1>Introduction to phylogenetics: tools and models</h1>
<!-- 3-11 -->
<p>Within the first year of the COVID-19 pandemic, almost 400,000 whole
or partial severe acute SARS-CoV-2 genomes were generated and shared
openly, marking an unprecedented worldwide response in pathogen genome
sequencing.</p>
<p>Phylogenetic tools have proven increasingly important in the public
health management of a variety of viral epidemics, but the COVID-19
pandemic is the first global health emergency in which large-scale,
real-time genome sequencing and analysis have guided public health
decisions. SARS-CoV-2 has been evolving at an estimated nucleotide
substitution rate ranging between <a
href="https://doi.org/10.1093/molbev/msaa314">10e−3 and 10e−4
substitutions per site and per year</a>. The global epidemiological and
virological situation changed constantly during the pandemic, and genome
sequence analysis was crucial in <a
href="https://doi.org/10.1038/s41576-022-00483-8">tracking the changing
scenario</a> (Figure 14).</p>
<div class="float">
<img src="images/OC4_3-11_fig1.png"
alt="Phylodynamic approaches to the investigation of SARS-CoV-2 transmission" />
<div class="figcaption">Phylodynamic approaches to the investigation of
SARS-CoV-2 transmission</div>
</div>
<p><em>Source: <a
href="https://doi.org/10.1038/s41576-022-00483-8">Nature Reviews
Genetics</a></em></p>
<p><a href="images/OC4_3-11_fig1.png">Click here to enlarge the
image</a></p>
<p><a href="images/OC4_3-11_Alt-text.pdf">Download Figure 14 and 15
alt-text here</a></p>
<p>Phylogenetic tools like <a
href="https://cov-lineages.org/resources/pangolin.html">Pangolin</a> and
<a href="https://clades.nextstrain.org/">Nextclade</a> can unlock
information from sampled genomes combined with epidemiological data
like:</p>
<ul>
<li>Quantifying international virus spread<br />
</li>
<li>Identifying outbreaks and transmission chains in specific
settings<br />
</li>
<li>Estimating growth rates and reproduction numbers<br />
</li>
<li>Identifying and tracking mutations of interest<br />
</li>
<li>Discovering and analysing variants of concern<br />
</li>
<li>Investigating intra-host virus evolution</li>
</ul>
<p>Different approaches to building the phylogenetic tree:</p>
<p>Parsimony approach - <a
href="https://doi.org/10.1186/1748-7188-7-9">Maximum Parsimony</a> is a
character-based approach that infers a phylogenetic tree by minimising
the total number of evolutionary steps required to explain a given set
of data assigned on the leaves. Tools that use parsimony include <a
href="http://iqtree.cibiv.univie.ac.at/">IQTREE</a> though it should be
noted this is a tool to generate ML trees and <a
href="https://genome.ucsc.edu/cgi-bin/hgPhyloPlace">USHER</a>.</p>
<p>Maximum likelihood approach- <a
href="http://scholarship.claremont.edu/scripps_theses/46">Maximum
Likelihood (ML)</a> is a method for the inference of phylogeny. It
evaluates a hypothesis about evolutionary history in terms of the
probability that the proposed model and the hypothesised history would
give rise to the observed data set. The supposition is that history with
a higher probability of reaching the observed state is preferred to
history with a lower probability. The method searches for the tree with
the highest probability or likelihood. ML is implemented on <a
href="http://www.microbesonline.org/fasttree/">FasTree</a>, <a
href="https://raxml-ng.vital-it.ch/#/">RAxML</a>.</p>
<p><strong>How to read a phylogenetic tree</strong></p>
<p>There are two parts to the phylogenetic tree: the nodes and the
branches (Figure15). The nodes on the tip of the tree represent the
cases having some ancestry that existed as a putative virus in an
individual. The internal nodes between the branches make up the ancestor
cases that are carrying the mutations from which further diversion was
possible. The branches represent the transmission chains, which are a
combination of numerous transmission events taking place and are
represented by the branch length.</p>
<div class="float">
<img src="images/OC4_3-11_fig2.png"
alt="Different graphic representations of a phylogenetic tree" />
<div class="figcaption">Different graphic representations of a
phylogenetic tree</div>
</div>
<p><a href="images/OC4_3-11_fig2.png">Click here to enlarge the
image</a></p>
<p>Bootstrapping: Bootstrap values in a phylogenetic tree indicate that
out of 100, how many times the same branch is observed when repeating
the generation of a phylogenetic tree on a resampled set of data. If we
get this observation 100 times out of 100, then this supports our
result.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-detect-an-outbreak" class="section level1">
<h1>How to detect an outbreak</h1>
<!-- 3-12 -->
<p><a href="https://www.cogconsortium.uk/">COG-UK</a> and similar
sequencing-based surveillance responses to the pandemic around the world
resulted in millions of genome sequences becoming available. As well as
the possibility of understanding the evolution of the virus and its
variants via phylogenetics, this wealth of data has presented
opportunities to study transmission dynamics at multiple levels,
including in <a
href="https://pubmed.ncbi.nlm.nih.gov/32463076/">healthcare
settings</a>, <a
href="https://www.nature.com/articles/s41467-021-27942-w">universities</a>,
<a
href="https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2789424">social
settings</a>, and <a
href="https://www.mdpi.com/1999-4915/14/3/545">wider
communities</a>.</p>
<p>‘Genomic epidemiology’ refers to the use of genomic data for tracking
viral transmission, combined with qualitative location data. Using this
method to study transmission dynamics may result in uncovering the
origin of an outbreak, routes of transmission and potential
super-spreading events. Super-spreading events have been a <a
href="https://elifesciences.org/articles/67308">significant concern
throughout the COVID-19 pandemic</a>, and they occur when SARS-CoV-2 is
transmitted <a href="https://pubmed.ncbi.nlm.nih.gov/26468744/">from one
individual to many other individuals</a>. Such events cause rapid spread
of the virus, so the ability to monitor them using genomic epidemiology
is essential for containing outbreaks and halting transmission chains as
much as possible.</p>
<p>Multiple sequence alignment-based bioinformatic tools can be used to
compare SARS-CoV-2 genome sequences and provide an idea of the level of
relatedness between cases. Whilst standard phylogenetic approaches
provide a general idea of relatedness and outbreak dynamics, they do not
enable the identification of directly linked cases, or the direction of
transmission events. Several tools have been developed to perform these
analyses in detail, with the aim of enabling more precise management of
outbreaks in hospital settings where <a
href="https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0268">nosocomial
transmission has been a major concern</a>.</p>
<p><a
href="https://academic.oup.com/mbe/article/36/3/587/5300248?login=true">Transcluster</a>
is a tool that was developed prior to the COVID-19 pandemic, modelled on
Mycobacterium tuberculosis transmission. This model of sequence
comparison enables the identification of ‘transmission clusters’ by
comparing differences in naturally accumulated single point genome
mutations - single nucleotide polymorphisms (SNPs) - within the
epidemiological context of positive cases. It incorporates factors such
as the time of testing positive and knowledge of the organism’s SNP
accumulation process to provide an accurate representation of closely
related cases. The tool produces a useful visual of transmission
clusters and the dynamics of relatedness between the sequences given to
the model (Figure 16). This can provide a clear idea of possible routes
of transmission, which is especially useful for managing hospital
outbreaks that spread across multiple wards and other locations.</p>
<div class="float">
<img src="images/OC4_3-12_fig1.png"
alt="Example of Transcluster plot simulating a COVID-19 outbreak that spread to four different locations within a building such as a hospital. Each circle represents an individual case and the colour of the circle represents the location where they tested positive. In this example, all cases belong to the same variant, but only those that lie within the blue oval are part of a transmission cluster based on the parameters of the model. The cases outside the oval were likely acquired outside of the hospital environment. A case that was identified in ‘Location C’ appeared to come into contact with many people in ‘Location A’, transmitting the virus to several people. This individual is classed as a ‘super-spreader’." />
<div class="figcaption">Example of Transcluster plot simulating a
COVID-19 outbreak that spread to four different locations within a
building such as a hospital. Each circle represents an individual case
and the colour of the circle represents the location where they tested
positive. In this example, all cases belong to the same variant, but
only those that lie within the blue oval are part of a transmission
cluster based on the parameters of the model. The cases outside the oval
were likely acquired outside of the hospital environment. A case that
was identified in ‘Location C’ appeared to come into contact with many
people in ‘Location A’, transmitting the virus to several people. This
individual is classed as a ‘super-spreader’.</div>
</div>
<p><a href="images/OC4_3-12_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-12_Alt-text.pdf">Download Figure 16 alt-text
here</a></p>
<p>Integrating genomic sequence data with epidemiological data provides
a detailed picture of outbreak origins and transmission dynamics. The
implementation of bioinformatic tools throughout the COVID-19 pandemic
resulted in specific transmission routes being uncovered, leading to
better real-time and prospective hospital infection control. In the
future, applying these tools in a more streamlined way during genomic
surveillance of problematic infectious disease agents is likely to
enable more rapid and specific outbreak management.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="generating-public-health-reports" class="section level1">
<h1>Generating public health reports</h1>
<!-- 3-13 -->
<p>Genomic epidemiology can be used to assist Infection Prevention
Control (IPC) teams by linking cases together that may otherwise be
missed, or by providing increased resolution to break down outbreaks
into distinct subgroups (Figure 17). Such subgroups may more easily
point towards sources of infection spread, such as; common treatment
regimes, interactions with individuals (e.g. healthcare workers or
visitors), or shared locations within the ward.</p>
<div class="float">
<img src="images/OC4_3-13_fig1.png"
alt="Infection of three different variants of SARS-CoV-2 amongst three wards. Whole genome sequencing (WGS) can identify distinct variants circulating in each ward, as well as identify closely linked cases that may be part of shared infection chains. Created with BioRender.com" />
<div class="figcaption">Infection of three different variants of
SARS-CoV-2 amongst three wards. Whole genome sequencing (WGS) can
identify distinct variants circulating in each ward, as well as identify
closely linked cases that may be part of shared infection chains.
Created with <a href="https://biorender.com/">BioRender.com</a></div>
</div>
<p><a href="images/OC4_3-13_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-13_Alt-text.pdf">Download Figure 17-19 alt-text
here</a></p>
<p><strong>Tools for reporting</strong></p>
<p>A number of freely available tools have been used by members of
COG-UK in order to analyse genomic data of SARS-CoV-2 in the context of
epidemiological assessment. One simple approach is to plot phylogenetic
trees to highlight the genetic distance (e.g. the number of nucleotides
that differ) between different samples. Cases of the same variant will
cluster on branches together, while other variants will sit on distinct
branches in the tree (Figure 18).</p>
<div class="float">
<img src="images/OC4_3-13_fig2.png"
alt="Example phylogenetic tree showing similarities and differences between sequenced cases of SARS-CoV-2. Cases with the same lineage, and thus fewer differences in genomic sequence, cluster together" />
<div class="figcaption">Example phylogenetic tree showing similarities
and differences between sequenced cases of SARS-CoV-2. Cases with the
same lineage, and thus fewer differences in genomic sequence, cluster
together</div>
</div>
<p><a href="images/OC4_3-13_fig2.png">Click to enlarge the image</a></p>
<p>In addition, tools such as <a
href="https://github.com/chjackson/a2bcovid">A2BCovid</a> and <a
href="https://github.com/reconhub/outbreaker2">OutBreaker2</a> and <a
href="https://github.com/JamesStimson/transcluster">TransCluster</a> can
be used to identify closely linked cases by factoring in the date of
sampling, the mutation rate of the virus, and the genomic similarity
between two cases.</p>
<p>The first two tools also allow users to incorporate epidemiological
data, such as ward locations of patients over their stay, to allow the
most likely disease outbreaks to be reconstructed. Similarly, the <a
href="https://cov-lineages.org/resources/civet.html">Cluster
Investigation and Virus Epidemiology Tool</a> (CIVET) was developed by
COG-UK to allow researchers to visualise new sequences alongside
previous cases from across the UK.</p>
<p>Such tools rapidly provide essential epidemiological context to new
cases and can thus help to better understand outbreaks. However, they
require in-depth computational experience and bioinformatics expertise
to use, which must be well documented in reports to stakeholders.</p>
<p><strong>Report writing</strong></p>
<p>To ensure that complex genomic data are of maximum benefit to IPC
teams, succinct reporting must be carried out in order to convey the key
information quickly and easily to stakeholders who may be
non-experts.</p>
<p>Reports must be generated rapidly to be of most benefit and provide
key information of benefit to IPC teams. This may include information on
variants currently circulating or known to have been circulating
previously, novel introductions of previously unseen variants,
identification of previously identified VOCs, and clustering of cases to
highlight potentially linked cases.</p>
<p>Consistent and regular reporting is important to ensure that IPC
teams can respond rapidly to the information. The use of reporting
templates, or even programmatically generated reports using tools such
as <a href="https://rmarkdown.rstudio.com/">R Markdown</a>, can help to
ensure consistency and are easily updated with additional analyses as
requirements develop. An example cluster plot from a weekly report from
the University of Portsmouth sequencing team is shown in Figure 19.</p>
<div class="float">
<img src="images/OC4_3-13_fig3.png"
alt="Example of R-markdown reporting of clusters identified using TransCluster. Potentially linked cases identified as clusters were further reported in a table to allow easy tracing by hospital staff" />
<div class="figcaption">Example of R-markdown reporting of clusters
identified using TransCluster. Potentially linked cases identified as
clusters were further reported in a table to allow easy tracing by
hospital staff</div>
</div>
<p><a href="images/OC4_3-13_fig3.png">Click to enlarge the image</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-prevent-sequencing-and-sampling-bias"
class="section level1">
<h1>How to prevent sequencing and sampling bias</h1>
<!-- 3-14 -->
<!-- YT https://youtu.be/rpMiDsoWaa4 -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/rpMiDsoWaa4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC4_3-14_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Catherine Ludden talks about sampling strategies
that can prevent biases which can impact public health actions.</p>
<p>When deciding which samples to sequence, there are many options
depending on the capacity of your network, the stage of the pandemic in
your country and the most pressing public health questions requiring
answers. Figure 20 shows some different categories of samples and one
way of thinking about prioritisation.</p>
<div class="float">
<img src="images/OC4_3-14_fig1.png"
alt="Example of potential categories of public health options and the types of samples for sequencing" />
<div class="figcaption">Example of potential categories of public health
options and the types of samples for sequencing</div>
</div>
<p><a href="images/OC4_3-14_fig1.png">Click to enlarge the image</a></p>
<p><a href="images/OC4_3-14_Alt-text.pdf">Donwload Figure 20 alt-text
here</a></p>
<p>Whilst these priorities will be different in each setting, it is
almost universally accepted that the most important and primary activity
in genomic surveillance is the tracking of different lineages and
variants. This is best achieved by taking a random, unbiased and
geographically representative sample of cases. This approach will be
informative as to the lineages circulating, potentially at low levels
depending on the size of the sample.</p>
<p>Sampling in hospitals, intensive care units, suspected imported
cases, and care homes can answer very important targeted public health
questions but, without being accompanied by general surveillance, risks
a bias in the data.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="using-sequencing-data-to-identify-outbreaks-in-a-hospital"
class="section level1">
<h1>Using sequencing data to identify outbreaks in a hospital</h1>
<!-- 3-15 -->
<!-- YT https://youtu.be/eg4mJKPjVeY -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/eg4mJKPjVeY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC4_3-15_transcript.pdf">Donwload the transcript
here</a></p>
<p>In this video call, Professor Judy Breuer discussed the application
of sequencing to investigate an outbreak in a hospital and its
importance to track nosocomial infection.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="the-importance-of-sample-and-data-consent"
class="section level1">
<h1>The importance of sample and data consent</h1>
<!-- 3-16 -->
<p>The major threat to global public health that SARS-CoV-2 presented
led to an unprecedented pace of much-needed research and vaccine
development. This accelerated both ethical procedures that inform
research and the performance of controversial studies (e.g. controlled
human infection studies - CHIS), where healthy people were infected with
COVID-19 so its mechanism of infection could be studied. Such research
raised concerns about the ethical, legal, social and clinical
implications of infecting healthy volunteers, because ‘due process’ in
research mandates consideration of the participant’s safety.</p>
<p>In general, genomics research involving human samples raises a number
of ethical challenges such as consent, privacy, and the collection,
storage and release of genomic data. In order to obtain valid consent
from individuals involved in genomics and medical research, including
SARS-CoV-2 and COVID-19 research, they must be adequately informed and
the research must be understood, undertaken voluntarily, and provided by
someone competent to do so.</p>
<p>Challenges exist in the designing and implementation of consent
processes in regions with lower literacy levels. The provision of
appropriate information in a comprehensible manner in genomics research
is a critical challenge, as concepts around “genetics”, “genomics” and
“data release” are complex to explain. Linking these concepts to more
accessible levels of knowledge (e.g. that facial traits are inherited in
families) may assist in narrowing this challenge. Nevertheless, ensuring
that all aspects of the genomics research are explained in an accessible
manner in order to obtain valid consent remains a critical obstacle.</p>
<p>Another ethical issue around sample and data consent is that the
information produced from this research may affect additional people
other than the participating individual. Privacy protection measures
exist where personal identifiers are removed from genomic data, lowering
the risk of revealing personal information. Yet, it is still possible
for potentially stigmatising information about populations, communities
or families to be revealed, which may have direct adverse effects on the
individuals that consented to be involved, and indirectly on those with
a relation to them.</p>
<p>These issues need to be addressed when designing consent processes
for genomics research, even when faced with periods of emergency like
the COVID-19 pandemic. Measures have been developed as guidelines
through the collaborative development of informed consent document (ICD)
templates. These are developed by and for genomics researchers in
consultation with ethics committees, whereby discussions on consent
procedures, challenges and best practices with researcher collaboration,
participant privacy and safety, samples and data storage etc., take
place.</p>
<p><strong>Ethics review</strong></p>
<p>Genomics research needs to obtain ethics approval from the relevant
committees prior to the commencement of research. The review of genomics
research can be challenging as the comprehension of the science may be
difficult; the nature of the studies being hypothesis-generating rather
than hypothesis-testing is complex. Teams must also consider how to
generate a large amount of data that can be analysed numerous times in
various ways, and which may also be shared amongst research groups.</p>
<p>These are the main points of consideration that have been raised by
ethics committees when assessing genomics studies:</p>
<ul>
<li>Ensuring participants give valid informed consent.</li>
<li>Justification of exporting samples and procedures pertaining to the
return or destruction of samples at the conclusion of the project.</li>
<li>Ensuring the appropriate recognition of collaborative or local
investigators and their contributions.</li>
<li>Ensuring no harm will be brought to participating individuals or
populations.</li>
<li>Ways to assign benefits to the collaborating researchers or
communities involved.</li>
</ul>
<p>There are additional significant issues to review when obtaining
valid consent for genomics research, which involves the importance of
privacy protection. All personal identifiers that link an individual to
a genomic dataset must be removed in order to reduce the risk of
identification and to ensure anonymity. When performing large-scale
collaborative research, such as the international efforts in the
COVID-19 pandemic, ethics approval from a multitude of global committees
is undoubtedly necessary. There are then major challenges in genomics
research due to the complexity of issues of consent and privacy, with
different continents, countries, states, and even institutions, having
varying requirements.</p>
<p><strong>Further reading</strong></p>
<p><a href="http://dx.doi.org/10.1136/medethics-2020-106504">SARS- CoV-
2 challenge studies: ethics and risk minimisation</a></p>
<p><a href="https://doi.org/10.5334/dsj-2017-049">Genomic research data
generation, analysis and sharing – challenges in the African
setting</a></p>
<p><a href="https://doi.org/10.1038/ejhg.2009.191">Ethical implications
of the use of whole-genome methods in medical research</a></p>
<p><a href="https://doi.org/10.1186/1472-6939-12-5">Ethical issues in
human genomics research in developing countries</a></p>
<p><a href="https://doi.org/10.1371/journal.pmed.0050192">Informed
Consent in the Genomics Era</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="establishing-collaborations-in-a-pandemic"
class="section level1">
<h1>Establishing collaborations in a pandemic</h1>
<!-- 3-17 -->
<!-- YT https://youtu.be/6hE-ZxUj2n8 -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/6hE-ZxUj2n8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC4_3-17_transcript.pdf">Download the transcript
here</a></p>
<p>How to collaborate and establish partnerships during a pandemic. Hear
the experiences of experts around the world in this video.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="challenges-and-how-to-overcome-them" class="section level1">
<h1>Challenges and how to overcome them</h1>
<!-- 3-18 -->
<!-- YT https://youtu.be/NkoxGIaoesI -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/NkoxGIaoesI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/Oc4_3-18_transcript.pdf">Download the transcript
here</a></p>
<p>In this interview, Paola Resende, Senjuti Saha, Murali Bashyam and
Christian Happy talk about the main challenges they faced during the
pandemic and how they overcame it.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="resources" class="section level1">
<h1>Resources</h1>
<!-- 3-20 -->
<p>Coronavirus biology: <a
href="https://viralzone.expasy.org/9056">ViralZone</a></p>
<p>Coronavirus taxonomy: <a
href="https://talk.ictvonline.org/taxonomy/">International Committee on
Taxonomy of Viruses (ICTV)</a></p>
<p><a
href="https://www.sumanasinc.com/scienceinfocus/sif_coronavirus.html">SARS-CoV-2
replication cycle</a> by Sultanas, Inc</p>
<p>Real-time tracking of pathogen evolution: <a
href="https://nextstrain.org/ncov/gisaid/global">NextStrain</a></p>
<p>Lineages nomenclature: <a
href="https://cov-lineages.org/index.html">Pango</a></p>
<p>Genomic data sharing: <a
href="https://www.gisaid.org/">GISAID</a></p>
<p>Cloud Infrastructure for Big Data Microbial Bioinformatics: <a
href="https://www.climb.ac.uk/">CLIMB</a></p>
<p>NCBI COVID-19 scientific literature: <a
href="https://www.ncbi.nlm.nih.gov/research/coronavirus/">LitCovid</a></p>
<p>SARS-CoV-2 sequencing protocol: <a
href="https://artic.network/ncov-2019">ARTIC protocol</a></p>
<p><strong>Examples of commercially available library prep kits or
open-source protocols used in SARS-CoV-2 WGS by
amplicon-seq</strong></p>
<p><a
href="https://emea.illumina.com/products/by-type/ivd-products/covidseq.html">CovidSeq</a></p>
<p><a href="https://doi.org/10.1186/s13073-021-00839-5">CoronaHiT:
high-throughput sequencing of SARS-CoV-2 genomes</a></p>
<p><a href="https://doi.org/10.1186/s12967-022-03307-9">Mini-XT, a
miniaturized tagmentation-based protocol for efficient sequencing of
SARS-CoV-2</a></p>
<p><a
href="https://international.neb.com/applications/ngs-sample-prep-and-target-enrichment/nebnext-artic-products-for-sars-cov-2-sequencing">NEBNext
ARTIC products for SARS-CoV-2 sequencing</a></p>
<p><a
href="https://www.protocols.io/view/ncov-2019-sequencing-protocol-v3-locost-bp2l6n26rgqe/v3">nCoV-2019
sequencing protocol v3 (LoCost)</a></p>
<p><a
href="https://www.protocols.io/view/sars-cov2-genome-sequencing-protocol-1200bp-amplic-rm7vz8q64vx1/v6">SARS-CoV-2
genome sequencing protocol (1200bp amplicon ‘midnight’ primer set, using
Nanopore Rapid kit)</a></p>
<p><a
href="https://www.pacb.com/research-focus/microbiology/public-health/covid-19-sequencing-tools-and-resources/">HifiViral
SARS-CoV-2 kit</a></p>
<p><a
href="https://www.thermofisher.com/uk/en/home/life-science/sequencing/dna-sequencing/microbial-sequencing/microbial-identification-ion-torrent-next-generation-sequencing/viral-typing/coronavirus-research.html">Ion
AmpliSeq SARS-CoV-2 Insight Research Assay</a></p>
<p><a
href="https://www.nimagen.com/fields-of-application/covid-19">EasySeq™
SARS-CoV-2 WGS kit (NimaGen, Nijmegen, The Netherlands)</a></p>
<p><strong>Examples of commercially available SARS-CoV-2 probe panels
used in probe/bait capture</strong></p>
<p><a
href="https://www.agilent.com/en/product/next-generation-sequencing/community-designs-ngs/infectious-disease-research-designs-536208">Agilent
Community Designs – Infectious Diseases</a></p>
<p><a
href="https://eu.idtdna.com/pages/products/next-generation-sequencing/workflow/xgen-ngs-hybridization-capture/pre-designed-hyb-cap-panels/sars-cov-2-hyb-panel">IDT
- xGen SARS-CoV-2 Hyb Panel</a></p>
<p><a
href="https://arborbiosci.com/genomics/targeted-sequencing/mybaits/mybaits-expert/mybaits-expert-virus-sars-cov-2/?gclid=CjwKCAjwq5-WBhB7EiwAl-HEkgJfoldddWDRiAJDchPVd-FOvnC_lu14x3ROSDcYf7VbvFgdPITu9hoC5LQQAvD_BwE">myBaits
Expert Virus SARS-CoV-2 panel (Arbor Biosciences)</a></p>
<p><a
href="https://www.illumina.com/products/by-type/sequencing-kits/library-prep-kits/respiratory-virus-oligo-panel.html">Illumina
RVOP</a></p>
<p><a
href="https://www.twistbioscience.com/products/ngs/fixed-panels/sars-cov-2-research-panel?tab=overview">SARS-CoV-2
Research Panel</a></p>
<p><a
href="https://sequencing.roche.com/en/products-solutions/products/sample-preparation/dna-reagents/target-enrichment/hybridization/kapa-te-custom-panel-covid-19.html">KAPA
HyperCap SARS-CoV-2 panel</a></p>
<p><a
href="https://emea.support.illumina.com/content/dam/illumina-support/documents/documentation/software_documentation/sav/sequencing-analysis-viewer-user-guide-15020619-f.pdf">Sequencing
Analysis Viewer Software User Guide</a></p>
<p><a
href="https://emea.illumina.com/systems/sequencing-platforms/miseq/specifications.html">MiSeq
Specifications</a></p>
<p><a
href="https://emea.illumina.com/systems/sequencing-platforms/nextseq/specifications.html">NextSeq
500/550 Specifications</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>

<!DOCTYPE html>
<html>
<head>
<style>
* {
  box-sizing: border-box;
}

.img-container-left {
  float: left;
  width: 33.33%;
  padding: 15px;
}

.img-container-right {
  float: right;
  width: 33.33%;
  padding: 15px;
}

.clearfix::after {
  content: "";
  clear: both;
  display: table;
}
</style>
</head>
<body>

<hr>
<div class="clearfix">
  <div class="img-container-left">
    <img src="assets/img/COG-Train_logo.png" alt="COG-Train" 
style="width:200%">
  </div>
  <div class="img-container-right">
    <img src="assets/img/WCS_Logo-Primary_Black.jpg" alt="Wellcome 
Connecting Science" style="width:200%">
  </div>
</div>

</body>
</html>





</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
