<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>swab_to_server_week3</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="assets/styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">COG-Train_Resources</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-file-lines"></span>
     
    Course Materials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">Virtual Courses</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Viral Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="vbioinfasia.html">VBioinf2022 Asia</a>
        </li>
        <li>
          <a href="vbioinflac.html">VBioinf2022 LAC</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">MOOCs</li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Power of Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="power_of_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="power_of_genomics_set1.html">Week 1</a>
        </li>
        <li>
          <a href="power_of_genomics_set2.html">Week 2</a>
        </li>
        <li>
          <a href="power_of_genomics_set3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Swab to Server</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="swab_to_server_home.html">Home</a>
        </li>
        <li>
          <a href="swab_to_server_week1.html">Week 1</a>
        </li>
        <li>
          <a href="swab_to_server_week2.html">Week 2</a>
        </li>
        <li>
          <a href="swab_to_server_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Making Sense</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="making_sense_home.html">Home</a>
        </li>
        <li>
          <a href="making_sense_week1.html">Week 1</a>
        </li>
        <li>
          <a href="making_sense_week2.html">Week 2</a>
        </li>
        <li>
          <a href="making_sense_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Practical Guide</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="practical_guide_home.html">Home</a>
        </li>
        <li>
          <a href="practical_guide_week1.html">Week 1</a>
        </li>
        <li>
          <a href="practical_guide_week2.html">Week 2</a>
        </li>
        <li>
          <a href="practical_guide_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="dropdown-submenu">
      <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">Pathogen Genomics</a>
      <ul class="dropdown-menu" role="menu">
        <li>
          <a href="pathogen_genomics_home.html">Home</a>
        </li>
        <li>
          <a href="pathogen_genomics_week1.html">Week 1</a>
        </li>
        <li>
          <a href="pathogen_genomics_week2.html">Week 2</a>
        </li>
        <li>
          <a href="pathogen_genomics_week3.html">Week 3</a>
        </li>
      </ul>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Hybrid Course</li>
    <li>
      <a href="scvo2b4b.html">SARS-CoV-2 B4B</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">In Person Courses</li>
    <li>
      <a href="ECPG.html">Pathogen Genomics Capacity</a>
    </li>
    <li>
      <a href="IGM_B4B.html">Bioinformática Decodificada</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Webinars</li>
    <li>
      <a href="Community.html">COG-Train Community</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-book"></span>
     
    Team
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cogteam.html">COG-Train Team</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/WCSCourses/COG-Train_Resources">
    <span class="fa fa-brands fa-github"></span>
     
    Github
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="a-sample-without-data-is-nothing" class="section level1">
<h1>A SAMPLE WITHOUT DATA IS NOTHING</h1>
<p><img src="images/OC2_W3_cover.png" /><br />
<sub><span class="citation">@Canva</span></sub></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="lessons-learnt-how-to-be-prepared-for-the-next-pandemic"
class="section level1">
<h1>Lessons learnt: how to be prepared for the next pandemic</h1>
<!-- 3-2 -->
<!-- YT https://youtu.be/xnvVYYQebyI -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/xnvVYYQebyI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC2_3-2_transcript.pdf">Download the transcript
here</a></p>
<p>Here, Sharon Glaysher, Sam Robson and Angie Beckett, from the
University of Portsmouth, review their experiences tackling the pandemic
so far and present us with the main challenges of sampling during the
pandemic and what they would do differently next time.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="data-sharing-in-peace-and-war-time" class="section level1">
<h1>Data sharing in “peace” and “war” time</h1>
<!-- 3-3 -->
<!-- YT https://youtu.be/b0seHMkm2i4 -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/b0seHMkm2i4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC2_3-3_transcript.pdf">Download the transcript
here</a></p>
<p>In this talk, Prof Senjuti Saha from the Child Health Research
Foundation in Bangladesh discusses her views on ethics and data sharing
during pandemics, or “wartime” and “peacetime” or periods between public
health emergencies.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="stakeholders-for-data-sharing-and-the-importance-to-share"
class="section level1">
<h1>Stakeholders for data sharing and the importance to share</h1>
<!-- 3-4 -->
<p><strong>Who is a stakeholder</strong></p>
<p>Sharing research data within the international scientific community
is of increasing importance for research propagation and value. However,
the ethical and social challenges this presents, particularly in the
context of structural inequities and varied capacity in international
research is of concern.</p>
<p>The involvement of the public and other stakeholders is essential to
building locally responsive research policies, including data sharing.
Stakeholders are individuals, groups, or organizations that are affected
by or can affect a particular action undertaken by others. Biobanks,
data repositories and other such central storage facilities or
organisations encompass a number of different stakeholders such as
donors, researchers, research institutions, regulatory bodies, funders,
and others. These stakeholders can potentially have a strong influence
on the organization and operation of any sample or data collection. A
sound strategy for stakeholder engagement is therefore essential.</p>
<p>As such, for more and less research-experienced stakeholders, ethical
research data sharing is likely to rest on the development and
implementation of appropriate trust-building processes, linked to local
perceptions of benefits and challenges.</p>
<p>Stakeholders involved in clinical research have different
roles/responsibilities in the process of data and sample sharing toward
the common goal of research and improving patient benefits. In general,
the sharing process (Figure 1) can be defined in a number of iterative
steps; donors providing data or samples to the collector; the collector
providing the samples and/or data to the sponsor, who stores them in a
database and/or biobank; data providers (sponsors of clinical study or
database or biobank); data provider making data or samples upfront
available, or requesters finding the data or material, requesting access
via an intermediary or directly to the provider, negotiating, and-upon
agreement-receiving the requested data or material by the requester.</p>
<div class="float">
<img src="images/OC2_3-4_Fig1.jpeg"
alt="Chain of stakeholders involved in the process of sharing clinical patient data and samples" />
<div class="figcaption">Chain of stakeholders involved in the process of
sharing clinical patient data and samples</div>
</div>
<p><em>Source: <a
href="https://doi.org/10.3389/fmed.2018.00006">Frontiers</a></em></p>
<p><a href="images/OC2_3-4_Fig1.jpeg">Click here to enlarge the
image</a></p>
<p><a href="images/OC2_3-4_Fig1_alt-text.pdf">Download Figure 1 alt-text
here</a></p>
<p><strong>Why share research data</strong></p>
<p>Research data are a valuable resource, usually requiring much time
and money to be produced. Many data have a significant value beyond
usage for the original research. Data sharing benefits the researcher,
research sponsors, data repositories, the scientific community, and the
public. It encourages more connection and collaboration between
scientists, and better science leads to better decision-making. Data
sharing is typically encouraged within the scientific community, but it
requires a great deal of effort, resources, and collaboration. Preparing
data to be shared takes time and careful documentation of the research
process and the data results. Sharing research data:</p>
<ul>
<li>Encourages scientific enquiry and debate<br />
</li>
<li>Promotes innovation and potential new data uses<br />
</li>
<li>Leads to new collaborations between data users and data
creators<br />
</li>
<li>Maximises transparency and accountability<br />
</li>
<li>Enables scrutiny of research findings<br />
</li>
<li>Encourages the improvement and validation of research methods<br />
</li>
<li>Reduces the cost of duplicating data collection<br />
</li>
<li>Increases the impact and visibility of research<br />
</li>
<li>Promotes the research that created the data and its outcomes<br />
</li>
<li>Can provide direct credit to the researcher as a research output in
its own right<br />
</li>
<li>Provides important resources for education and training</li>
</ul>
<p>The ease with which digital data can be stored, disseminated and made
easily accessible online to users means that many institutions are keen
to share research data to increase the impact and visibility of their
research.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="opportunities-for-global-collaboration-in-pathogen-genomics"
class="section level1">
<h1>Opportunities for global collaboration in pathogen genomics</h1>
<!-- 3-5 -->
<!-- YT https://youtu.be/JrAwSESZQo0 -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/JrAwSESZQo0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC2_3-5_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Ewan Harrison, from COG-UK, talks about the
potential opportunities of pathogen genomics for global scientific and
health benefits.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="challenges-and-limitations-of-data-sharing"
class="section level1">
<h1>Challenges and limitations of data sharing</h1>
<!-- 3-6 -->
<!-- YT https://youtu.be/5ImhT7NJPHs -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/5ImhT7NJPHs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/Oc2_3-6_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Ewan Harrison tells his experiences in COG-UK
dealing with legal considerations when sharing data among partners from
different countries.</p>
<p><strong>Obstacles to the release of data</strong></p>
<p>We are living in an unprecedented era of information, leading to
progress for open access to science and global research data. Funders
around the world are increasingly mandating good data practices, such as
data management plans and data sharing, and recognizing the importance
of global collaboration on infrastructure and best practices. Across the
research community, policy, strategy, and working groups are building
momentum toward a future in which research data is widely Findable,
Accessible, Interoperable, and Reusable (FAIR). In this section, we
discuss a variety of challenges associated with data sharing, such as
obstacles to the release of data, privacy and confidentiality issues,
and informed-consent issues.</p>
<p>There are a variety of challenges associated with data release. Some
are related to the concerns of the scientists who generated the data,
some are related to the concerns of businesses or other organizations
that paid for the data collection, and some are practical issues related
to data administration.</p>
<p><strong>Concerns About Adversarial Science</strong></p>
<p>Regrettably, research can be highly contentious and confrontational
at times. For example, industries that profit from the production or use
of specific chemicals or goods may oppose research and/or data
indicating that their specific chemicals or goods pose health risks.</p>
<p><strong>Business Considerations Related to Data Sharing</strong></p>
<p>When deciding whether to share data, businesses always consider a
variety of factors. Two of the most significant are concerns about
exposing themselves to liability and other costs, as well as concerns
about losing the value of confidential business information.</p>
<p><strong>The Business Value of Data</strong></p>
<p>There is also the question of who pays for access to data that a
company has paid for. Regulatory agencies are generally required to
safeguard the commercial value of data collected by businesses. The most
important fact is that the data generated by industry researchers have
commercial value. Companies that collected data deserve and expect to be
compensated for supplying that data to others; however, where should
that compensation come from? If another company wants to use the data,
the answer is obvious; however, if academic researchers want to use the
data, the answer is less clear.</p>
<p><strong>Administrative Issues</strong></p>
<p>Data sharing is also complicated by a number of administrative
issues, one of which is who owns the data. This is especially
problematic when data has been collected by groups of companies or
institutions. Other administrative challenges include the fact that
organizational policies and procedures, as well as logistical issues,
may differ from one organization to the next. For example, different
universities’ institutional review boards may have different rules
governing data sharing procedures.</p>
<p><strong>The cost of Data Sharing</strong></p>
<p>The cost of extensive data sharing is a major impediment. The issue
is that most budgets do not cover the costs of data sharing. If data are
to be made publicly available, the funds to do so must come from
somewhere, perhaps from direct research support.</p>
<p><strong>Privacy and confidentiality issues</strong></p>
<p>One of the barriers to data release is ensuring the privacy of the
persons whose data has been obtained as much as possible, as some of
these data, such as medical history data or employment data, can be
highly sensitive. Organizations are putting plans to share data on hold
due to concerns over re-identification. A key question is how likely the
re-identification of subjects in existing data sets is that have had the
obvious personal identifying information removed.</p>
<p><strong>Informed-consent issues</strong></p>
<p>Before an individual can take part in a scientific or medical
research study, it is generally necessary for that person to provide
“informed consent.” The problem is that when you ask someone for
consent, especially broad consent, neither the requestor nor the person
being sought has any idea what that means in terms of how it will be
utilized because we don’t even know what it will be used for down the
road. The second point is that when people asked for consent a long time
ago, we knew that consent meant you weren’t going to provide anything to
someone else, but as long as we removed your name from it, it was
fine.</p>
<p><strong>Insufficient tools and technology</strong></p>
<p>The technological challenges highlighted include transferring large
datasets, particularly to the African region. Furthermore, technology is
viewed as simplifying data-related operations rather than adding
complexity. However, many alternative cloud data providers are now
accessible, each with their own set of data access rules. As data teams
continue to rapidly embrace numerous cloud data platforms, this
heterogeneous patchwork of capabilities frequently fails to scale
effectively across different cloud data platforms. As a consequence,
either too tight data rules that prevent data exchange entirely, or
overly wide policies that allow sensitive data to slip through the gaps,
resulting in a data breach or leak.</p>
<p><strong>Further reading</strong></p>
<p><a href="https://doi.org/10.12688/aasopenres.12968.1">Ethical and
practical issues to consider in the governance of genomic and human
research data and data sharing in South Africa: a meeting report</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="data-sharing-best-practices" class="section level1">
<h1>Data sharing best practices</h1>
<!-- 3-9 -->
<audio controls>
<source src="assets/OC2_3-9_audio.mp3" type="audio/ogg">
</audio>
<p><a href="assets/OC2_3-9_transcript.pdf">Donwload the transcript
here</a></p>
<p>In this audio clip, you will listen to Newton Lwanga and Maria
Magdalene Namaganda from Makerere University, Uganda talking about the
current data-sharing repositories and the best practices to share
research data.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="the-fair-principles" class="section level1">
<h1>The FAIR principles</h1>
<!-- 3-10 -->
<p>It is widely acknowledged that sharing research data in a timely and
open manner is crucial to furthering progress in science and healthcare.
However, due to the diverse nature of different research approaches,
systems, national guidance and the field-specific practice, merely
making data available does not always maximise the utility to the global
community.</p>
<p>With these issues in mind, a group of researchers hosted a
stakeholder workshop with experts from academia and the private sector
in 2014 to discuss how to address the challenges. The workshop generated
a draft set of principles guiding the sharing of data. They agreed that
all research output should be Findable, Accessible, Interoperable and
Reusable (FAIR). These <a
href="https://www.nature.com/articles/sdata201618#Sec6"><strong>FAIR</strong>
guiding principles</a> were subsequently expanded upon as below.</p>
<p>To be <strong>Findable</strong>:<br />
F1. (meta)data are assigned a globally unique and persistent
identifier.</p>
<p>F2. data are described with rich metadata (defined by R1 below)</p>
<p>F3. metadata clearly and explicitly include the identifier of the
data it describes.</p>
<p>F4. (meta)data are registered or indexed in a searchable
resource.</p>
<p>To be <strong>Accessible</strong>:</p>
<p>A1. (meta)data are retrievable by their identifier using a
standardized communications protocol.</p>
<p>A1.1 the protocol is open, free, and universally implementable.</p>
<p>A1.2 the protocol allows for an authentication and authorization
procedure, where necessary.</p>
<p>A2. metadata are accessible, even when the data are no longer
available.</p>
<p>To be <strong>Interoperable</strong>:</p>
<p>I1. (meta)data use a formal, accessible, shared and broadly
applicable language for knowledge representation.</p>
<p>I2. (meta)data use vocabularies that follow FAIR principles.</p>
<p>I3. (meta)data include qualified references to other (meta)data.</p>
<p>To be <strong>Reusable</strong>:</p>
<p>R1. meta(data) are richly described with a plurality of accurate and
relevant attributes.</p>
<p>R1.1. (meta)data are released with a clear and accessible data usage
license.</p>
<p>R1.2. (meta)data are associated with detailed provenance.</p>
<p>R1.3. (meta)data meet domain-relevant community standards</p>
<p>These principles were published in 2016 and have been implemented
widely across many fields since then. During the current pandemic, a
number of groups have sought to consider how to implement these
principles with the additional fast-paced pressure of an ongoing
outbreak.</p>
<p>The <a href="https://www.nature.com/articles/s41431-020-0635-7">Virus
Outbreak Data Network (<strong>VODAN</strong>)</a> established a project
to develop an international data network infrastructure supporting
evidence-based responses to the pandemic. One of the projects associated
with VODAN has shown that this approach can be <a
href="https://www.medrxiv.org/content/10.1101/2021.08.13.21262023v1">implemented
in a hospital setting</a> to allow rapid linking and sharing of relevant
patient information in a machine-readable format. A further project
highlighted the limited genomic data emerging from Africa underpinned by
concerns around data ownership and availability of health data at the
point of care. A system was developed to provide this data at the point
of care as well as aggregated for global analysis and has been <a
href="https://onlinelibrary.wiley.com/doi/full/10.1002/ggn2.10050">deployed
in a number of African countries</a>.</p>
<p>As well as the FAIR principles for data sharing, The Research Data
Alliance International Indigenous Data Sovereignty Interest Group
created a set of principles concerning indigenous data governance. These
principles intend to maximise the benefits of data and research on
individual communities, whilst promoting the outward sharing of data for
global good in a manner which respects previous barriers to sharing and
misuse of data. These principles state that there should be Collective
benefit, Authority to control, Responsibility and Ethics <a
href="https://www.nature.com/articles/s41597-021-00892-0#Fig2">(CARE)</a>.
There have been subsequent efforts to guide the use of both FAIR and
CARE guidance in data usage (Table 1).</p>
<p><em>Table 1 - Implementation of the CARE Principles across the data
lifecycle</em></p>
<table>
<colgroup>
<col width="25%" />
<col width="25%" />
<col width="25%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>Practice CARE in data collection</th>
<th>Engage CARE in data stewardship</th>
<th>Implement CARE in data community</th>
<th>Use FAIR and CARE in data applications</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Define cultural data; record provenance in metadata</td>
<td>Use appropriate governance models; make data FAIR</td>
<td>Indigenous ethics inform access; use tools for transparency,
integrity and provenance</td>
<td>Fairness, accountability, transparency; access equity</td>
</tr>
</tbody>
</table>
<p><em>Source: <a
href="https://doi.org/10.1038/s41597-021-00892-0">Scientific
Data</a></em></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-do-fair-principles-apply-in-a-public-health-emergency"
class="section level1">
<h1>How do FAIR principles apply in a public health emergency</h1>
<!-- 3-11 -->
<!-- YT https://youtu.be/tbjo1cLNgQI -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/tbjo1cLNgQI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/Oc2_3-11_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Dr Ewan Harrison from COG-UK guides us through the
differences between data sharing routinely and during a public health
emergency.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="no-database-is-perfect-beware-of-errors-in-sequence-databases"
class="section level1">
<h1>No database is perfect: beware of errors in sequence databases</h1>
<!-- 3-12 -->
<p>Public and private initiatives to implement and extend the sequencing
capability on a global basis, particularly in developing countries are
helping to find blind spots of SARS-COV-2 genetic information in Latin
America and the Caribbean and Africa. For instance, Brazil has increased
its sequencing capacity from both public and private initiatives and
today is among the 20 countries that most contribute with SARS-COV-2
sequences in <a
href="https://www.gisaid.org/about-us/public-private-partnerships/">GISAID
(global data science initiative) database</a>.</p>
<p>However, an effective genomic surveillance system requires not only
wide sampling but also an effective effort to share the data generated
truthfully and accurately in public databases to allow researchers to
track in real-time the variants and their mutations and to perform
epidemiological and biological studies.</p>
<p>As the COVID-19 pandemic evolves, the public databases have been
inundated with a massive amount of data, and SARS-COV-2 genomic
information is mostly available at three main databases: <a
href="https://www.ncbi.nlm.nih.gov/genbank/">NCBI GenBank</a>; <a
href="https://db.cngb.org/gisaid/">GISAID’s EpiCoV Database</a> and <a
href="https://www.covid19dataportal.org/">EMBL-EBI COVID-19 Data
Portal</a>. The raw reads of the sequencing can be also deposited in
proper databases such as <a href="https://www.insdc.org/">The
International Nucleotide Sequence Database Collaboration (INSDC)</a>.
Automated pipelines for quick deposition and processing of new genomic
information have become available and help researchers to submit really
large files quicker and easier than before.</p>
<p>Before releasing a new sequence, all submitted data are reviewed and
curated by a team of curators. Also, some basic information about the
sequences is mandatory in all databases, such as the name of the virus,
the sampling local, and the name and contact of the submitter. While we
can submit the raw data in some databases, at the GISAID platform, the
most accessed database for COVID-19 genomic information, only the final
assembly (not the raw reads) can be deposited. It turns out that the
automated assembly of reads always involves an interpretation of
inevitable errors generated during the sequencing process. The
sequencing and assembly processes can also be biased due to technique
artefacts.</p>
<p>In most cases, the errors are relatively minor such as a couple of
nucleotide substitutions (“sequencing artefacts”) or non-covered regions
that are replaced by a sequence of “NNN”. In other cases, however, the
errors are more significant, and the use of such data can lead to
erroneous interpretations and even inconsistent conclusions. As an
example, while frameshift mutations are easily detected in an assembled
genome (frameshift: where one or more nucleotides are inserted or
deleted in a way that changes the triplet reading code), missense
substitutions or in-frame indels (that involves at least 3 or multiple
of 3 nucleotides and do not change the reading frame) are almost
impossible to be recognized. Last, but not least, there are also
“annotation” errors. There is no easy way to detect misclassifications
as an incorrect geographic location of sampling, sampling dating, or
even clinical information associated with the sequences.</p>
<p>We are in the era of automation, the era of next-generation
sequencing and bioinformatics advances, but it is still our
responsibility to assure the quality of the data we generate. Robust
Quality control processes are required prior to uploading to data
repositories and caution should be taken when using data obtained from
such sources. Any surprising or impactful findings should always
consider the possibility of inaccurate sequence or meta-data before
publication.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-establish-fruitful-and-fair-collaboration-an-example"
class="section level1">
<h1>How to establish fruitful and fair collaboration: an example</h1>
<!-- 3-13 -->
<!-- YT https://youtu.be/DpJCq8dR32Q -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/DpJCq8dR32Q" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC2_3-13_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Prof Mark Webber and Dr Muhammad Yasir, from the
Quadram Institute, talk about their collaboration to tackle COVID-19 in
Pakistan. They also share some tips and ideas for those who are looking
for new collaborations.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="how-to-promote-data-sharing" class="section level1">
<h1>How to promote data sharing</h1>
<!-- 3-14 -->
<p>How to ask for cooperation for global data sharing? Here Dr Ewan
Harrison talks about the COG-UK approach.</p>
<p>Ewan talked about a couple of different advantages the UK had but the
underlying concepts are generalisable when trying to promote data
sharing:</p>
<p><strong>Be trustworthy</strong></p>
<p>Whether you are looking to engage new research collaborators, seek
data from different communities or aggregate data from many different
sources you have to demonstrate that you are a safe and trustworthy
custodian of that data. Secure storage and transmission of data, robust
data management plans, ethical approvals and transparency will all help
to achieve this aim.</p>
<p><strong>Listen</strong></p>
<p>Many of the reasons why individuals, communities or organisations are
reluctant to share data are grounded in previous issues. Listen to what
your colleagues are concerned about. Do they fear a lack of credit for
their contributions, are they concerned their own community won’t
benefit from the research or do they worry about the perception of
sharing data with external institutions? Understanding their concerns is
likely to be a constructive step towards beginning to address them and
eventually agreeing to share.</p>
<p><strong>Work collaboratively</strong></p>
<p>Agree on roles and responsibilities together. Who will hold the data?
Who is going to write the outputs? How will contributions be recognised?
Early agreement on a number of key steps will prevent miscommunication
and future disagreements.</p>
<p><strong>Further reading</strong></p>
<p><a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02395-y">CLIMB-COVID:
continuous integration supporting decentralised sequencing for
SARS-CoV-2 genomic surveillance</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="importance-of-implementing-local-capacity"
class="section level1">
<h1>Importance of implementing local capacity</h1>
<!-- 3-15 -->
<!-- YT https://youtu.be/gEQEy_AXjBQ -->
<iframe width="840" height="472" src="https://www.youtube.com/embed/gEQEy_AXjBQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
</iframe>
<p><a href="assets/OC2_3-15_transcript.pdf">Download the transcript
here</a></p>
<p>In this video, Sharon Glaysher, Sam Robson, and Angie Beckett from
Portsmouth University share their experiences over the emergence of the
Alpha variant. They highlight the importance of having laboratory
capacity and contingency plans in place.</p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>
<div id="resources" class="section level1">
<h1>Resources</h1>
<!-- 3-18 -->
<p>Coronavirus biology: <a
href="https://viralzone.expasy.org/9056">ViralZone</a><br />
Coronavirus taxonomy: <a
href="https://talk.ictvonline.org/taxonomy/">International Committee on
Taxonomy of Viruses (ICTV)</a><br />
Real-time tracking of pathogen evolution: <a
href="https://nextstrain.org/ncov/gisaid/global">NextStrain</a><br />
Lineages nomenclature: <a
href="https://cov-lineages.org/index.html">Pango</a><br />
Genomic data sharing: <a href="https://www.gisaid.org/">GISAID</a><br />
Cloud Infrastructure for Big Data Microbial Bioinformatics: <a
href="https://www.climb.ac.uk/">CLIMB</a><br />
NCBI COVID-19 scientific literature: <a
href="https://www.ncbi.nlm.nih.gov/research/coronavirus/">LitCovid</a></p>
<hr style="height:5px;border-width:0;color:gray;background-color:gray">
</div>

<!DOCTYPE html>
<html>
<head>
<style>
* {
  box-sizing: border-box;
}

.img-container-left {
  float: left;
  width: 33.33%;
  padding: 15px;
}

.img-container-right {
  float: right;
  width: 33.33%;
  padding: 15px;
}

.clearfix::after {
  content: "";
  clear: both;
  display: table;
}
</style>
</head>
<body>

<hr>
<div class="clearfix">
  <div class="img-container-left">
    <img src="assets/img/COG-Train_logo.png" alt="COG-Train" 
style="width:200%">
  </div>
  <div class="img-container-right">
    <img src="assets/img/WCS_Logo-Primary_Black.jpg" alt="Wellcome 
Connecting Science" style="width:200%">
  </div>
</div>

</body>
</html>





</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
